diff --git a/core/edo.ttl b/core/edo.ttl
index 6237c2a..e34649e 100644
--- a/core/edo.ttl
+++ b/core/edo.ttl
@@ -675,32 +675,8 @@ edo:hasExternalRef rdf:type owl:AnnotationProperty ;
                    dcterms:identifier "hasExternalRef" .
 
 
-###  https://w3id.org/energy-domain/edo#ifc_equivalentClass
-edo:ifc_equivalentClass rdf:type owl:AnnotationProperty ;
-                        rdfs:subPropertyOf edo:DomainInteroperabilityAnnotation ;
-                        rdfs:label "Classe IFC equivalente"@pt-br ,
-                                   "IFC equivalent Class"@en ;
-                        skos:definition "Classe IFC equivalente"@pt-br ,
-                                        "IFC equivalent class"@en .
 
 
-###  https://w3id.org/energy-domain/edo#ifc_objectType
-edo:ifc_objectType rdf:type owl:AnnotationProperty ;
-                   rdfs:subPropertyOf edo:DomainInteroperabilityAnnotation ;
-                   rdfs:label "IFC ObjectType"@en ,
-                              "ObjectType IFC"@pt-br ;
-                   skos:definition "Object Type for IFC class customization"@en ,
-                                   "Object Type para a customização de classes IFC"@pt-br .
-
-
-###  https://w3id.org/energy-domain/edo#ifc_predefinedType
-edo:ifc_predefinedType rdf:type owl:AnnotationProperty ;
-                       rdfs:subPropertyOf edo:DomainInteroperabilityAnnotation ;
-                       rdfs:label "IFC PredefinedType"@en ,
-                                  "PredefinedType IFC"@pt-br ;
-                       skos:definition "Predefined Type for IFC class customization"@en ,
-                                       "Predefined Type para a customização de classes IFC"@pt-br .
-
 
 ###  https://w3id.org/energy-domain/edo#hasMaturityLevel
 edo:hasMaturityLevel rdf:type owl:AnnotationProperty ;
@@ -753,9 +729,7 @@ edo:AbandonmentCap rdf:type owl:Class ;
                                    "Temporary device used to seal the end of a cable, preventing water ingress."@en ;
                    skos:prefLabel "Abandonment Cap"@en ,
                                   "Capa de Abandono"@pt-br ;
-                   edo:hasDiscipline edo:SubseaUmbilicalsEngineering ;
-                   edo:ifc_equivalentClass "IfcCableFitting" ;
-                   edo:ifc_objectType "AbandonmentCap" .
+                   edo:hasDiscipline edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#AbrasionProtector
@@ -768,9 +742,7 @@ edo:AbrasionProtector rdf:type owl:Class ;
                                       "Componente aplicado sobre cabos ou dutos flexíveis para reduzir desgaste por atrito e contato com superfícies, aumentando a vida útil do sistema."@pt-br ;
                       skos:prefLabel "Abrasion Protector"@en ,
                                      "Protetor Anti-Abrasivo"@pt-br ;
-                      edo:hasDiscipline edo:SubseaUmbilicalsEngineering ;
-                      edo:ifc_equivalentClass "IfcProtectiveDevice" ;
-                      edo:ifc_objectType "AbrasionProtector" .
+                      edo:hasDiscipline edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#AbsoluteInsidePressure
@@ -871,10 +843,7 @@ edo:AnchoringCollar rdf:type owl:Class ;
                                      edo:MetallicStrandSpareQuantity ,
                                      edo:SafeWorkingLoad ;
                     edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ,
-                                      edo:SubseaUmbilicalsEngineering ;
-                    edo:ifc_equivalentClass "IfcPipeFitting" ;
-                    edo:ifc_objectType "AnchoringCollar" ;
-                    edo:ifc_predefinedType "USERDEFINED" .
+                                      edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#AnodeCollarSet
@@ -890,10 +859,7 @@ edo:AnodeCollarSet rdf:type owl:Class ;
                    skos:prefLabel "Anode Collar Set"@en ,
                                   "Conjunto de Colares de Anodo"@pt-br ;
                    edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ,
-                                     edo:SubseaUmbilicalsEngineering ;
-                   edo:ifc_equivalentClass "IfcPipeFitting" ;
-                   edo:ifc_objectType "AnodeCollarSet" ;
-                   edo:ifc_predefinedType "USERDEFINED" .
+                                     edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#AnodeCollarsAxialSpacing
@@ -941,10 +907,7 @@ edo:AntiCollapseSheath rdf:type owl:Class ;
                                        "Reinforcement layer in subsea flexible pipes used in oil and gas exploration, designed to prevent the collapse of the pipe under high external pressure conditions, such as those encountered in deepwater environments. Its primary role is to provide structural support against external forces that could compress or deform the pipe, particularly during situations where the internal pressure is reduced or absent, like when the pipe is emptied. Typically made from high-strength metallic or polymeric materials, the Anti Collapse Sheath ensures the integrity of the flexible pipe under extreme conditions."@en ;
                        skos:prefLabel "Anti Collapse Sheath"@en ,
                                       "Capa Anti-Colapso"@pt-br ;
-                       edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ;
-                       edo:ifc_equivalentClass "IfcPipeSegment" ;
-                       edo:ifc_objectType "AntiCollapseSheath" ;
-                       edo:ifc_predefinedType "USERDEFINED" .
+                       edo:hasDiscipline edo:SubseaFlexiblePipesEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#AntiWearSheath
@@ -957,10 +920,7 @@ edo:AntiWearSheath rdf:type owl:Class ;
                    skos:prefLabel "Anti Wear Sheath"@en ,
                                   "Capa Anti-Atrito"@pt-br ;
                    edo:hasAttribute edo:TapesQuantity ;
-                   edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ;
-                   edo:ifc_equivalentClass "IfcPipeSegment" ;
-                   edo:ifc_objectType "AntiWearSheath" ;
-                   edo:ifc_predefinedType "USERDEFINED" .
+                   edo:hasDiscipline edo:SubseaFlexiblePipesEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#AnyURIType
@@ -992,10 +952,7 @@ edo:ArmorPot rdf:type owl:Class ;
                              "Umbilical armor termination element"@en ;
              skos:prefLabel "Armor Pot"@en ,
                             "Armor Pot"@pt-br ;
-             edo:hasDiscipline edo:SubseaUmbilicalsEngineering ;
-             edo:ifc_equivalentClass "IfcPipeFitting" ;
-             edo:ifc_objectType "ArmorPot" ;
-             edo:ifc_predefinedType "USERDEFINED" .
+             edo:hasDiscipline edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#ArmourLayer
@@ -1276,9 +1233,7 @@ edo:BendRestrictor rdf:type owl:Class ;
                                     edo:BendMomentVsShearForceTable_Condition ,
                                     edo:BendMomentVsShearForceTable_ShearForce ;
                    edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ,
-                                     edo:SubseaUmbilicalsEngineering ;
-                   edo:ifc_equivalentClass "IfcPipeFitting" ;
-                   edo:ifc_objectType "BendRestrictor" .
+                                     edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#BendStiffener
@@ -1440,10 +1395,7 @@ edo:BuoyancyModule rdf:type owl:Class ;
                                     edo:ShortTermFloatDensity ,
                                     edo:ShortTermFloatVolume ;
                    edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ,
-                                     edo:SubseaUmbilicalsEngineering ;
-                   edo:ifc_equivalentClass "IfcPipeFitting" ;
-                   edo:ifc_objectType "BuoyancyModule" ;
-                   edo:ifc_predefinedType "USERDEFINED" .
+                                     edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#BuoyancyTank
@@ -1623,10 +1575,7 @@ edo:CommissioningActivity rdf:type owl:Class ;
                           dcterms:identifier "CommissioningActivity" ;
                           skos:prefLabel "Atividade de Comissionamento"@pt-br ,
                                          "Commissioning Activity"@en ;
-                          edo:hasAttribute edo:Planned_Start_Timestamp ;
-                          edo:ifc_equivalentClass "IfcTask" ;
-                          edo:ifc_objectType "Activity" ;
-                          edo:ifc_predefinedType "USERDEFINED" .
+                          edo:hasAttribute edo:Planned_Start_Timestamp .
 
 
 ###  https://w3id.org/energy-domain/edo#CommissioningContract
@@ -1634,10 +1583,7 @@ edo:CommissioningContract rdf:type owl:Class ;
                           rdfs:subClassOf edo:CommissioningElement ;
                           dcterms:identifier "CommissioningContract" ;
                           skos:prefLabel "Commissioning Contract"@en ,
-                                         "Contrato de Comissionamento"@pt-br ;
-                          edo:ifc_equivalentClass "IfcGroup" ;
-                          edo:ifc_objectType "Contract" ;
-                          edo:ifc_predefinedType "USERDEFINED" .
+                                         "Contrato de Comissionamento"@pt-br .
 
 
 ###  https://w3id.org/energy-domain/edo#CommissioningDigitalProcessStep
@@ -1645,10 +1591,7 @@ edo:CommissioningDigitalProcessStep rdf:type owl:Class ;
                                     rdfs:subClassOf edo:CommissioningElement ;
                                     dcterms:identifier "CommissioningDigitalProcessStep" ;
                                     skos:prefLabel "Commissioning Digital Process Step"@en ,
-                                                   "Passo do Processo Digital"@pt-br ;
-                                    edo:ifc_equivalentClass "IfcTask" ;
-                                    edo:ifc_objectType "CommissioningDigitalProcessStep" ;
-                                    edo:ifc_predefinedType "USERDEFINED" .
+                                                   "Passo do Processo Digital"@pt-br .
 
 
 ###  https://w3id.org/energy-domain/edo#CommissioningElement
@@ -1666,10 +1609,7 @@ edo:CommissioningEvidence rdf:type owl:Class ;
                           rdfs:subClassOf edo:CommissioningElement ;
                           dcterms:identifier "CommissioningEvidence" ;
                           skos:prefLabel "Commissioning Evidence"@en ,
-                                         "Evidência de Comissionamento"@pt-br ;
-                          edo:ifc_equivalentClass "IfcDocumentInformation" ;
-                          edo:ifc_objectType "CommissioningEvidence" ;
-                          edo:ifc_predefinedType "USERDEFINED" .
+                                         "Evidência de Comissionamento"@pt-br .
 
 
 ###  https://w3id.org/energy-domain/edo#CommissioningIssue
@@ -1677,10 +1617,7 @@ edo:CommissioningIssue rdf:type owl:Class ;
                        rdfs:subClassOf edo:CommissioningElement ;
                        dcterms:identifier "CommissioningIssue" ;
                        skos:prefLabel "Commissioning Issue"@en ,
-                                      "Pendência de Comissionamento"@pt-br ;
-                       edo:ifc_equivalentClass "IfcClassification" ;
-                       edo:ifc_objectType "Pendency_Classification" ;
-                       edo:ifc_predefinedType "USERDEFINED" .
+                                      "Pendência de Comissionamento"@pt-br .
 
 
 ###  https://w3id.org/energy-domain/edo#CommissioningItemCheck
@@ -1688,10 +1625,7 @@ edo:CommissioningItemCheck rdf:type owl:Class ;
                            rdfs:subClassOf edo:CommissioningElement ;
                            dcterms:identifier "CommissioningItemCheck" ;
                            skos:prefLabel "Commissioning Item Check"@en ,
-                                          "Verificação de Item Comissionável"@pt-br ;
-                           edo:ifc_equivalentClass "IfcClassificationReference" ;
-                           edo:ifc_objectType "FVI" ;
-                           edo:ifc_predefinedType "USERDEFINED" .
+                                          "Verificação de Item Comissionável"@pt-br .
 
 
 ###  https://w3id.org/energy-domain/edo#CommissioningLoopCheck
@@ -1699,10 +1633,7 @@ edo:CommissioningLoopCheck rdf:type owl:Class ;
                            rdfs:subClassOf edo:CommissioningElement ;
                            dcterms:identifier "CommissioningLoopCheck" ;
                            skos:prefLabel "Commissioning Loop Check"@en ,
-                                          "Verificação de Malha"@pt-br ;
-                           edo:ifc_equivalentClass "IfcClassificationReference" ;
-                           edo:ifc_objectType "FVM" ;
-                           edo:ifc_predefinedType "USERDEFINED" .
+                                          "Verificação de Malha"@pt-br .
 
 
 ###  https://w3id.org/energy-domain/edo#CommissioningPerson
@@ -1710,10 +1641,7 @@ edo:CommissioningPerson rdf:type owl:Class ;
                         rdfs:subClassOf edo:CommissioningElement ;
                         dcterms:identifier "CommissioningPerson" ;
                         skos:prefLabel "Commissioning Person"@en ,
-                                       "Pessoa"@pt-br ;
-                        edo:ifc_equivalentClass "IfcPerson" ;
-                        edo:ifc_objectType "CommissioningPerson" ;
-                        edo:ifc_predefinedType "USERDEFINED" .
+                                       "Pessoa"@pt-br .
 
 
 ###  https://w3id.org/energy-domain/edo#CommissioningPreservationOrder
@@ -1721,10 +1649,7 @@ edo:CommissioningPreservationOrder rdf:type owl:Class ;
                                    rdfs:subClassOf edo:CommissioningElement ;
                                    dcterms:identifier "CommissioningPreservationOrder" ;
                                    skos:prefLabel "Commissioning Preservation Order"@en ,
-                                                  "Ordem de Preservação"@pt-br ;
-                                   edo:ifc_equivalentClass "IfcClassificationReference" ;
-                                   edo:ifc_objectType "OP" ;
-                                   edo:ifc_predefinedType "USERDEFINED" .
+                                                  "Ordem de Preservação"@pt-br .
 
 
 ###  https://w3id.org/energy-domain/edo#CommissioningProgram
@@ -1732,10 +1657,7 @@ edo:CommissioningProgram rdf:type owl:Class ;
                          rdfs:subClassOf edo:CommissioningElement ;
                          dcterms:identifier "CommissioningProgram" ;
                          skos:prefLabel "Commissioning Program"@en ,
-                                        "Programa de Comissionamento"@pt-br ;
-                         edo:ifc_equivalentClass "IfcGroup" ;
-                         edo:ifc_objectType "Program" ;
-                         edo:ifc_predefinedType "USERDEFINED" .
+                                        "Programa de Comissionamento"@pt-br .
 
 
 ###  https://w3id.org/energy-domain/edo#CommissioningProject
@@ -1743,10 +1665,7 @@ edo:CommissioningProject rdf:type owl:Class ;
                          rdfs:subClassOf edo:CommissioningElement ;
                          dcterms:identifier "CommissioningProject" ;
                          skos:prefLabel "Commissioning Project"@en ,
-                                        "Projeto de Comissionamento"@pt-br ;
-                         edo:ifc_equivalentClass "IfcProject" ;
-                         edo:ifc_objectType "Project" ;
-                         edo:ifc_predefinedType "USERDEFINED" .
+                                        "Projeto de Comissionamento"@pt-br .
 
 
 ###  https://w3id.org/energy-domain/edo#CommissioningResponsibleActor
@@ -1754,10 +1673,7 @@ edo:CommissioningResponsibleActor rdf:type owl:Class ;
                                   rdfs:subClassOf edo:CommissioningElement ;
                                   dcterms:identifier "CommissioningResponsibleActor" ;
                                   skos:prefLabel "Ator Responsável"@pt-br ,
-                                                 "Commissioning Responsible Actor"@en ;
-                                  edo:ifc_equivalentClass "IfcActor" ;
-                                  edo:ifc_objectType "CommissioningResponsibleActor" ;
-                                  edo:ifc_predefinedType "USERDEFINED" .
+                                                 "Commissioning Responsible Actor"@en .
 
 
 ###  https://w3id.org/energy-domain/edo#CommissioningResponsibleGroup
@@ -1765,10 +1681,7 @@ edo:CommissioningResponsibleGroup rdf:type owl:Class ;
                                   rdfs:subClassOf edo:CommissioningElement ;
                                   dcterms:identifier "CommissioningResponsibleGroup" ;
                                   skos:prefLabel "Commissioning Responsible Group"@en ,
-                                                 "Grupo Responsável"@pt-br ;
-                                  edo:ifc_equivalentClass "IfcGroup" ;
-                                  edo:ifc_objectType "CommissioningResponsibleGroup" ;
-                                  edo:ifc_predefinedType "USERDEFINED" .
+                                                 "Grupo Responsável"@pt-br .
 
 
 ###  https://w3id.org/energy-domain/edo#CommissioningTask
@@ -1780,10 +1693,7 @@ edo:CommissioningTask rdf:type owl:Class ;
                       edo:hasAttribute edo:Content ,
                                        edo:Creator ,
                                        edo:CreatorId ,
-                                       edo:Timestamp ;
-                      edo:ifc_equivalentClass "IfcTask" ;
-                      edo:ifc_objectType "Task" ;
-                      edo:ifc_predefinedType "USERDEFINED" .
+                                       edo:Timestamp .
 
 
 ###  https://w3id.org/energy-domain/edo#CompactObject
@@ -1855,9 +1765,7 @@ edo:CompositeMaterial rdf:type owl:Class ;
                                        edo:UltTensStrAlongFibers ,
                                        edo:UltTensStrPerpendFibers ;
                       edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ,
-                                        edo:SubseaUmbilicalsEngineering ;
-                      edo:ifc_equivalentClass "IfcMaterial" ;
-                      edo:ifc_objectType "CompositeMaterial" .
+                                        edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#ConceptualDesign
@@ -2130,10 +2038,7 @@ edo:DeadweightCollar rdf:type owl:Class ;
                      skos:prefLabel "Colar de Peso Morto"@pt-br ,
                                     "Deadweight Collar"@en ;
                      edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ,
-                                       edo:SubseaUmbilicalsEngineering ;
-                     edo:ifc_equivalentClass "IfcPipeFitting" ;
-                     edo:ifc_objectType "DeadweightCollar" ;
-                     edo:ifc_predefinedType "USERDEFINED" .
+                                       edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#DecimalValue
@@ -2311,9 +2216,7 @@ edo:DimensionsDrawing rdf:type owl:Class ;
                                       "Technical drawing providing detailed measurements, geometric specifications, and angular data of components. The Dimensions Drawing includes critical information such as length, diameter, thickness, and angles, ensuring the precise fabrication, assembly, and installation of the parts. This drawing is essential for meeting design requirements and ensuring that all components fit accurately within the overall system, supporting efficient manufacturing and quality control."@en ;
                       skos:prefLabel "Desenho de Cotas"@pt-br ,
                                      "Dimensions Drawing"@en ;
-                      edo:hasDiscipline edo:SubseaUmbilicalsEngineering ;
-                      edo:ifc_equivalentClass "IfcDocumentInformation" ;
-                      edo:ifc_objectType "DimensionsDrawing" .
+                      edo:hasDiscipline edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#DimensionsTableColumn
@@ -2756,10 +2659,7 @@ edo:ElectricalCable rdf:type owl:Class ;
                                    "Cabo elétrico"@pt-br ,
                                    "Electrical Cable"@en ,
                                    "Electrical cable"@en ;
-                    edo:hasDiscipline edo:SubseaUmbilicalsEngineering ;
-                    edo:ifc_equivalentClass "IfcCableSegment" ;
-                    edo:ifc_objectType "ElectricalCable" ;
-                    edo:ifc_predefinedType "USERDEFINED" .
+                    edo:hasDiscipline edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#ElectricalEngineering
@@ -2789,9 +2689,7 @@ edo:ElectricalJumperConnector rdf:type owl:Class ;
                                               "Connector for termination of electrical signal/low-current jumpers or cables."@en ;
                               skos:prefLabel "Conector de Jumper Elétrico"@pt-br ,
                                              "Electrical Jumper Connector"@en ;
-                              edo:hasDiscipline edo:SubseaUmbilicalsEngineering ;
-                              edo:ifc_equivalentClass "IfcCableFitting" ;
-                              edo:ifc_objectType "ElectricalJumperConnector" .
+                              edo:hasDiscipline edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#ElectricalPowerJumper
@@ -2810,9 +2708,7 @@ edo:ElectricalPowerJumperConnector rdf:type owl:Class ;
                                                    "Connector for termination of high-voltage/high-current electrical jumpers or cables."@en ;
                                    skos:prefLabel "Conector de Jumper Elétrico de Potência"@pt-br ,
                                                   "Electrical Power Jumper Connector"@en ;
-                                   edo:hasDiscipline edo:SubseaUmbilicalsEngineering ;
-                                   edo:ifc_equivalentClass "IfcCableFitting" ;
-                                   edo:ifc_objectType "ElectricalPowerJumperConnector" .
+                                   edo:hasDiscipline edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#ElectronicDevice
@@ -2853,10 +2749,7 @@ edo:EndFitting rdf:type owl:Class ;
                                 edo:RingGasketSpecification ,
                                 edo:RingGasketStandard ,
                                 edo:RingGasketType ;
-               edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ;
-               edo:ifc_equivalentClass "IfcPipeFitting" ;
-               edo:ifc_objectType "EndFitting" ;
-               edo:ifc_predefinedType "USERDEFINED" .
+               edo:hasDiscipline edo:SubseaFlexiblePipesEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#EnergyStorage
@@ -2994,10 +2887,7 @@ edo:FabricTape rdf:type owl:Class ;
                skos:prefLabel "Fabric Tape"@en ,
                               "Fita Construtiva"@pt-br ;
                edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ,
-                                 edo:SubseaUmbilicalsEngineering ;
-               edo:ifc_equivalentClass "IfcPipeSegment" ;
-               edo:ifc_objectType "FabricTape" ;
-               edo:ifc_predefinedType "USERDEFINED" .
+                                 edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#Fabrication
@@ -3154,9 +3044,7 @@ edo:FlangeAdapter rdf:type owl:Class ;
                   skos:prefLabel "Adaptador de Flange"@pt-br ,
                                  "Flange Adapter"@en ;
                   edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ,
-                                    edo:SubseaUmbilicalsEngineering ;
-                  edo:ifc_equivalentClass "IfcPipeFitting" ;
-                  edo:ifc_objectType "FlangeAdapter" .
+                                    edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#FlangeAttribute
@@ -3173,10 +3061,7 @@ edo:FlangeConnection rdf:type owl:Class ;
                                      "Represents a logical fluid port connection defined by a flange interface. It indicates the precise logical location within a model where fluid distribution components are connected, ensuring correct alignment and functionality of the system during assembly and installation. This connection type defines the interface between fluid distribution elements in subsea or other fluid-handling systems."@en ;
                      skos:prefLabel "Conexão Flangeada"@pt-br ,
                                     "Flange Connection"@en ;
-                     edo:hasDiscipline edo:SubseaUmbilicalsEngineering ;
-                     edo:ifc_equivalentClass "IfcDistributionPort" ;
-                     edo:ifc_objectType "FlangeConnection" ;
-                     edo:ifc_predefinedType "USERDEFINED" .
+                     edo:hasDiscipline edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#FlangeFaceType
@@ -3264,10 +3149,7 @@ edo:FlexiblePipeSegment rdf:type owl:Class ;
                                          edo:StrIntOnNoTensNomPress ,
                                          edo:StrIntOnNoTensNomPressValRef ,
                                          edo:StrIntOnNoTensNomPressValRefMult ;
-                        edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ;
-                        edo:ifc_equivalentClass "IfcPipeSegment" ;
-                        edo:ifc_objectType "FlexiblePipeSegment" ;
-                        edo:ifc_predefinedType "USERDEFINED" .
+                        edo:hasDiscipline edo:SubseaFlexiblePipesEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#FlexiblePipeStructure
@@ -3335,10 +3217,7 @@ edo:FlexiblePipeStructure rdf:type owl:Class ;
                                            edo:TestConditions ,
                                            edo:ThermalExchangeCoefficientFlooded ,
                                            edo:UtilizationFactor ;
-                          edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ;
-                          edo:ifc_equivalentClass "IfcPipeSegmentType" ;
-                          edo:ifc_objectType "FlexiblePipeStructure" ;
-                          edo:ifc_predefinedType "USERDEFINED" .
+                          edo:hasDiscipline edo:SubseaFlexiblePipesEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#FlexibleStructure
@@ -3430,10 +3309,7 @@ edo:FloatingProductionUnit rdf:type owl:Class ;
                                            "Plataforma offshore usada na exploração e produção de petróleo e gás, projetada para extrair, processar e armazenar hidrocarbonetos. A Unidade de Produção Flutuante (FPU) é ancorada ou amarrada ao leito marinho e pode operar em águas profundas, frequentemente conectada a poços submarinos por meio de dutos flexíveis e umbilicais. Ela acomoda equipamentos de processamento para separação de petróleo, gás e água, bem como instalações para armazenamento temporário ou transferência de hidrocarbonetos processados para navios-tanque ou dutos. As FPUs são essenciais para a produção eficiente e segura de petróleo e gás em ambientes offshore desafiadores."@pt-br ;
                            skos:prefLabel "Floating Production Unit"@en ,
                                           "Unidade de Produção Flutuante (UEP)"@pt-br ;
-                           edo:hasDiscipline edo:SubseaUmbilicalsEngineering ;
-                           edo:ifc_equivalentClass "IfcBuilding" ;
-                           edo:ifc_objectType "FloatingProductionUnit" ;
-                           edo:ifc_predefinedType "USERDEFINED" .
+                           edo:hasDiscipline edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#FlowBase
@@ -3715,10 +3591,7 @@ edo:GroovePoint rdf:type owl:Class ;
                                  edo:SafeWorkingLoad ,
                                  edo:TechnicalNotes ;
                 edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ,
-                                  edo:SubseaUmbilicalsEngineering ;
-                edo:ifc_equivalentClass "IfcDistributionPort" ;
-                edo:ifc_objectType "GroovePoint" ;
-                edo:ifc_predefinedType "USERDEFINED" .
+                                  edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#GrooveSupportSurfaceDiameter
@@ -3814,10 +3687,7 @@ edo:HCRHose rdf:type owl:Class ;
                             "Represents a high-pressure rubber hose used within tubing for fluid transport in umbilical systems."@en ;
             skos:prefLabel "HCR Hose"@en ,
                            "Mangueira HCR"@pt-br ;
-            edo:hasDiscipline edo:SubseaUmbilicalsEngineering ;
-            edo:ifc_equivalentClass "IfcPipeSegment" ;
-            edo:ifc_objectType "HCRHose" ;
-            edo:ifc_predefinedType "USERDEFINED" .
+            edo:hasDiscipline edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#HPHousing
@@ -3850,10 +3720,7 @@ edo:HandlingCollar rdf:type owl:Class ;
                    skos:prefLabel "Colar de Manuseio"@pt-br ,
                                   "Handling Collar"@en ;
                    edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ,
-                                     edo:SubseaUmbilicalsEngineering ;
-                   edo:ifc_equivalentClass "IfcPipeFitting" ;
-                   edo:ifc_objectType "HandlingCollar" ;
-                   edo:ifc_predefinedType "USERDEFINED" .
+                                     edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#HangOffCollar
@@ -3875,10 +3742,7 @@ edo:HangOffCollar rdf:type owl:Class ;
                                    edo:SafeWorkingLoad ,
                                    edo:UpperItubeDiameter ;
                   edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ,
-                                    edo:SubseaUmbilicalsEngineering ;
-                  edo:ifc_equivalentClass "IfcPipeFitting" ;
-                  edo:ifc_objectType "HangOffCollar" ;
-                  edo:ifc_predefinedType "USERDEFINED" .
+                                    edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#HardwareItem
@@ -3968,10 +3832,7 @@ edo:HighStrengthTape rdf:type owl:Class ;
                      skos:definition "Material used in subsea flexible pipes, typically applied as a reinforcing layer to provide additional strength and resistance to external forces. High Strength Tape is designed to improve the tensile properties of the flexible pipe, enhancing its ability to withstand high internal pressures, tension, and dynamic loads in harsh subsea environments. It is typically made from advanced fibers or composite materials, ensuring durability and long-term performance in demanding conditions."@en ;
                      skos:prefLabel "Fita de Alta Resistência"@pt-br ,
                                     "High Strength Tape"@en ;
-                     edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ;
-                     edo:ifc_equivalentClass "IfcPipeSegment" ;
-                     edo:ifc_objectType "HighStrengthTape" ;
-                     edo:ifc_predefinedType "USERDEFINED" .
+                     edo:hasDiscipline edo:SubseaFlexiblePipesEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#HistoricalAttribute
@@ -3995,10 +3856,7 @@ edo:HoldingBandage rdf:type owl:Class ;
                    edo:hasAttribute edo:ModulusOfElasticity ,
                                     edo:StressAtDesignPressure ,
                                     edo:TapesQuantity ;
-                   edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ;
-                   edo:ifc_equivalentClass "IfcPipeSegment" ;
-                   edo:ifc_objectType "HoldingBandage" ;
-                   edo:ifc_predefinedType "USERDEFINED" .
+                   edo:hasDiscipline edo:SubseaFlexiblePipesEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#Hook
@@ -4018,9 +3876,7 @@ edo:HotStab rdf:type owl:Class ;
             skos:prefLabel "Hot Stab"@en ,
                            "Hot Stab"@pt-br ;
             edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ,
-                              edo:SubseaUmbilicalsEngineering ;
-            edo:ifc_equivalentClass "IfcPipeFitting" ;
-            edo:ifc_objectType "HotStab" .
+                              edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#HotStabReceptacle
@@ -4032,9 +3888,7 @@ edo:HotStabReceptacle rdf:type owl:Class ;
                       skos:prefLabel "Hot Stab Receptacle"@en ,
                                      "Receptáculo de Hot Stab"@pt-br ;
                       edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ,
-                                        edo:SubseaUmbilicalsEngineering ;
-                      edo:ifc_equivalentClass "IfcPipeFitting" ;
-                      edo:ifc_objectType "HotStabReceptacle" .
+                                        edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#Hub
@@ -4195,10 +4049,7 @@ edo:IfcInstanciableElement rdf:type owl:Class ;
                            skos:definition "Representa um elemento de domínio que pode ser instanciado em projetos IFC, fornecendo anotações de configuração para suas instâncias específicas e permitindo a gestão consistente de propriedades entre múltiplas instâncias."@pt-br ,
                                            "Represents a domain element that can be instantiated in IFC projects, providing configuration annotations for its specific instances and enabling consistent property management across multiple instances."@en ;
                            skos:prefLabel "Elemento Instanciável IFC"@pt-br ,
-                                          "IFC Instanciable Element"@en ;
-                           edo:ifc_equivalentClass "-" ;
-                           edo:ifc_objectType "-" ;
-                           edo:ifc_predefinedType "-" .
+                                          "IFC Instanciable Element"@en .
 
 
 ###  https://w3id.org/energy-domain/edo#ImportedValue
@@ -4263,10 +4114,7 @@ edo:InnerTube rdf:type owl:Class ;
               skos:definition "The Inner Tube is the innermost layer of a subsea flexible pipe, responsible for containing and transporting the fluid, such as oil or gas. It is designed to withstand internal pressures and prevent fluid leakage while offering resistance to the corrosive and abrasive nature of the transported substances. The Inner Tube is typically made from high-performance polymeric materials that ensure durability, flexibility, and chemical resistance, making it a critical component in maintaining the integrity of the flexible pipe in subsea operations."@en ;
               skos:prefLabel "Inner Tube"@en ,
                              "Tubo Interno"@pt-br ;
-              edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ;
-              edo:ifc_equivalentClass "IfcPipeSegment" ;
-              edo:ifc_objectType "InnerTube" ;
-              edo:ifc_predefinedType "USERDEFINED" .
+              edo:hasDiscipline edo:SubseaFlexiblePipesEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#InsideTemperature
@@ -4329,10 +4177,7 @@ edo:InsulationTape rdf:type owl:Class ;
                    skos:definition "Material used in subsea flexible pipes to provide thermal insulation, ensuring that the temperature of the transported fluids remains stable during flow. Insulation Tape is applied in layers around the pipe to minimize heat loss or gain, which is critical for maintaining the fluid's properties, such as viscosity, in deepwater environments. It is typically made from materials with low thermal conductivity, such as polymers or composites, and is essential for enhancing the efficiency and performance of subsea pipelines in harsh temperature conditions."@en ;
                    skos:prefLabel "Fita de Isolamento"@pt-br ,
                                   "Insulation Tape"@en ;
-                   edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ;
-                   edo:ifc_equivalentClass "IfcPipeSegment" ;
-                   edo:ifc_objectType "InsulationTape" ;
-                   edo:ifc_predefinedType "USERDEFINED" .
+                   edo:hasDiscipline edo:SubseaFlexiblePipesEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#IntValue
@@ -4357,10 +4202,7 @@ edo:IntegratedPipe rdf:type owl:Class ;
                                   "Integrated Pipe"@en ,
                                   "Integrated pipe"@en ,
                                   "Tubo Integrado"@pt-br ;
-                   edo:hasDiscipline edo:SubseaUmbilicalsEngineering ;
-                   edo:ifc_equivalentClass "IfcPipeSegment" ;
-                   edo:ifc_objectType "IntegratedPipe" ;
-                   edo:ifc_predefinedType "USERDEFINED" .
+                   edo:hasDiscipline edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#Interconnection
@@ -4381,10 +4223,7 @@ edo:InterlockedCarcass rdf:type owl:Class ;
                        skos:definition "The Interlocked Carcass is a metallic layer used in certain subsea flexible pipes to provide internal pressure resistance and prevent collapse. It is designed with interlocking metal strips that offer high strength while maintaining flexibility. This layer enhances the structural integrity of the flexible pipe, protecting it from external pressures and harsh subsea conditions. The Interlocked Carcass allows the pipe to bend without losing its shape or collapsing under deepwater pressure."@en ;
                        skos:prefLabel "Carcaça"@pt-br ,
                                       "Interlocked Carcass"@en ;
-                       edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ;
-                       edo:ifc_equivalentClass "IfcPipeSegment" ;
-                       edo:ifc_objectType "InterlockedCarcass" ;
-                       edo:ifc_predefinedType "USERDEFINED" .
+                       edo:hasDiscipline edo:SubseaFlexiblePipesEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#IntermediateBendStiffener
@@ -4400,9 +4239,7 @@ edo:IntermediateBendStiffener rdf:type owl:Class ;
                               skos:prefLabel "Enrijecedor de Curvatura Intermediário"@pt-br ,
                                              "Intermediate Bend Stiffener"@en ;
                               edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ,
-                                                edo:SubseaUmbilicalsEngineering ;
-                              edo:ifc_equivalentClass "IfcPipeFitting" ;
-                              edo:ifc_objectType "IntermediateBendStiffener" .
+                                                edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#IntermediateSheath
@@ -4414,10 +4251,7 @@ edo:IntermediateSheath rdf:type owl:Class ;
                        skos:prefLabel "Barreira Intermediária de Vedação"@pt-br ,
                                       "Intermediate Sheath"@en ;
                        edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ,
-                                         edo:SubseaUmbilicalsEngineering ;
-                       edo:ifc_equivalentClass "IfcPipeSegment" ;
-                       edo:ifc_objectType "IntermediateSheath" ;
-                       edo:ifc_predefinedType "USERDEFINED" .
+                                         edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#IntermediateValuesStrategy
@@ -4526,10 +4360,7 @@ edo:InternalPressureSheath rdf:type owl:Class ;
                            skos:prefLabel "Barreira Interna de Pressão"@pt-br ,
                                           "Internal Pressure Sheath"@en ;
                            edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ,
-                                             edo:SubseaUmbilicalsEngineering ;
-                           edo:ifc_equivalentClass "IfcPipeSegment" ;
-                           edo:ifc_objectType "InternalPressureSheath" ;
-                           edo:ifc_predefinedType "USERDEFINED" .
+                                             edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#InternalVolume
@@ -5095,10 +4926,7 @@ edo:LowResistanceTape rdf:type owl:Class ;
                       skos:definition "A layer used in subsea flexible pipes to reduce friction between different components or layers, enhancing the overall flexibility and durability of the pipe. The Low Resistance Tape minimizes wear and abrasion caused by the movement of adjacent layers during operation, particularly in dynamic subsea conditions. Made from materials with low friction properties, it helps extend the lifespan of the pipe by reducing internal stresses and preventing damage to critical layers, ensuring smooth and efficient operation under harsh underwater conditions."@en ;
                       skos:prefLabel "Fita de Baixa Resistência"@pt-br ,
                                      "Low Resistance Tape"@en ;
-                      edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ;
-                      edo:ifc_equivalentClass "IfcPipeSegment" ;
-                      edo:ifc_objectType "LowResistanceTape" ;
-                      edo:ifc_predefinedType "USERDEFINED" .
+                      edo:hasDiscipline edo:SubseaFlexiblePipesEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#Lubricant
@@ -5628,9 +5456,7 @@ edo:MetallicMaterial rdf:type owl:Class ;
                                       edo:MetallicLayerMaterialType ,
                                       edo:UltimateTensileStress ;
                      edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ,
-                                       edo:SubseaUmbilicalsEngineering ;
-                     edo:ifc_equivalentClass "IfcMaterial" ;
-                     edo:ifc_objectType "MetallicMaterial" .
+                                       edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#MetallicStrandLength
@@ -5659,10 +5485,7 @@ edo:MetallicStrandSet rdf:type owl:Class ;
                                        edo:StrandsTable_StrandsLength ,
                                        edo:StrandsTable_StrandsQuantity ;
                       edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ,
-                                        edo:SubseaUmbilicalsEngineering ;
-                      edo:ifc_equivalentClass "IfcCableSegment" ;
-                      edo:ifc_objectType "MetallicStrandSet" ;
-                      edo:ifc_predefinedType "USERDEFINED" .
+                                        edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#MetallicStrandSpareQuantity
@@ -5690,10 +5513,7 @@ edo:MetallicTubing rdf:type owl:Class ;
                                   "Metallic tubing"@en ,
                                   "Tubulação Metálica"@pt-br ,
                                   "Tubulação metálica"@pt-br ;
-                   edo:hasDiscipline edo:SubseaUmbilicalsEngineering ;
-                   edo:ifc_equivalentClass "IfcPipeSegment" ;
-                   edo:ifc_objectType "MetallicTubing" ;
-                   edo:ifc_predefinedType "USERDEFINED" .
+                   edo:hasDiscipline edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#MetricBoltSet
@@ -5709,10 +5529,7 @@ edo:MetricBoltSet rdf:type owl:Class ;
                                    edo:ThreadPitch ,
                                    edo:ThreadStandard ;
                   edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ,
-                                    edo:SubseaUmbilicalsEngineering ;
-                  edo:ifc_equivalentClass "IfcMechanicalFastener" ;
-                  edo:ifc_objectType "MetricBoltSet" ;
-                  edo:ifc_predefinedType "USERDEFINED" .
+                                    edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#MetricStudSet
@@ -5729,10 +5546,7 @@ edo:MetricStudSet rdf:type owl:Class ;
                                    edo:ThreadPitch ,
                                    edo:ThreadStandard ;
                   edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ,
-                                    edo:SubseaUmbilicalsEngineering ;
-                  edo:ifc_equivalentClass "IfcMechanicalFastener" ;
-                  edo:ifc_objectType "MetricStudSet" ;
-                  edo:ifc_predefinedType "USERDEFINED" .
+                                    edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#Midstream
@@ -6174,10 +5988,7 @@ edo:OpticalFiberCable rdf:type owl:Class ;
                                      "Cabo de fibra ótica"@pt-br ,
                                      "Optical Fiber Cable"@en ,
                                      "Optical fiber cable"@en ;
-                      edo:hasDiscipline edo:SubseaUmbilicalsEngineering ;
-                      edo:ifc_equivalentClass "IfcCableSegment" ;
-                      edo:ifc_objectType "OpticalFiberCable" ;
-                      edo:ifc_predefinedType "USERDEFINED" .
+                      edo:hasDiscipline edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#OuterDiameter
@@ -6474,10 +6285,7 @@ edo:PipePullingHead rdf:type owl:Class ;
                                      edo:RingGasketStandard ,
                                      edo:RingGasketType ,
                                      edo:SafeWorkingLoad ;
-                    edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ;
-                    edo:ifc_equivalentClass "IfcPipeFitting" ;
-                    edo:ifc_objectType "PipePullingHead" ;
-                    edo:ifc_predefinedType "USERDEFINED" .
+                    edo:hasDiscipline edo:SubseaFlexiblePipesEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#PipeSectionApplication
@@ -6575,10 +6383,7 @@ edo:PlasticHose rdf:type owl:Class ;
                 skos:prefLabel "Mangueira Plástica"@pt-br ,
                                "Mangueira plástica"@pt-br ,
                                "Plastic Hose"@en ;
-                edo:hasDiscipline edo:SubseaUmbilicalsEngineering ;
-                edo:ifc_equivalentClass "IfcPipeSegment" ;
-                edo:ifc_objectType "PlasticHose" ;
-                edo:ifc_predefinedType "USERDEFINED" .
+                edo:hasDiscipline edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#PointLocation
@@ -6650,9 +6455,7 @@ edo:PolymericMaterial rdf:type owl:Class ;
                                        edo:SolubilityTable_SolubilityCoefficient ,
                                        edo:SolubilityTable_Temperature ;
                       edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ,
-                                        edo:SubseaUmbilicalsEngineering ;
-                      edo:ifc_equivalentClass "IfcMaterial" ;
-                      edo:ifc_objectType "PolymericMaterial" .
+                                        edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#Port
@@ -6719,10 +6522,7 @@ edo:PressureArmour rdf:type owl:Class ;
                    skos:definition "A high-strength, metallic layer used in subsea flexible pipes to provide resistance against both internal and external pressure. The Pressure Armour prevents the pipe from expanding under high internal pressures and protects against collapse due to external pressure. Typically made from helically wound steel or other strong materials, it supports the pipe's structural integrity and ensures reliable operation in harsh subsea environments."@en ;
                    skos:prefLabel "Armadura de Pressão"@pt-br ,
                                   "Pressure Armour"@en ;
-                   edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ;
-                   edo:ifc_equivalentClass "IfcPipeSegment" ;
-                   edo:ifc_objectType "PressureArmour" ;
-                   edo:ifc_predefinedType "USERDEFINED" .
+                   edo:hasDiscipline edo:SubseaFlexiblePipesEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#PressureComponent
@@ -6804,8 +6604,7 @@ edo:Project rdf:type owl:Class ;
             skos:definition "A broad concept that encompasses the design, development, and planning of various components or systems."@en ;
             skos:prefLabel "Project"@en ,
                            "Projeto"@pt-br ;
-            edo:hasDiscipline edo:SubseaUmbilicalsEngineering ;
-            edo:ifc_equivalentClass "IfcProject" .
+            edo:hasDiscipline edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#ProjectAcronym
@@ -6876,10 +6675,7 @@ edo:ProtectiveSheath rdf:type owl:Class ;
                      skos:prefLabel "Capa de Proteção"@pt-br ,
                                     "Protective Sheath"@en ;
                      edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ,
-                                       edo:SubseaUmbilicalsEngineering ;
-                     edo:ifc_equivalentClass "IfcPipeSegment" ;
-                     edo:ifc_objectType "ProtectiveSheath" ;
-                     edo:ifc_predefinedType "USERDEFINED" .
+                                       edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#ProvenanceAttribute
@@ -6903,10 +6699,7 @@ edo:PullInCollar rdf:type owl:Class ;
                                   edo:WireRopeSlingDiameter ,
                                   edo:WireRopeSlingLength ;
                  edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ,
-                                   edo:SubseaUmbilicalsEngineering ;
-                 edo:ifc_equivalentClass "IfcPipeFitting" ;
-                 edo:ifc_objectType "PullInCollar" ;
-                 edo:ifc_predefinedType "USERDEFINED" .
+                                   edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#PullingHead
@@ -6986,9 +6779,7 @@ edo:QuickConnectCoupling rdf:type owl:Class ;
                                          "Quick connect coupling used to connect or disconnect fluid or gas lines in a practical and safe way, ensuring sealing and minimising leakage during operation."@en ;
                          skos:prefLabel "Engate Rápido"@pt-br ,
                                         "Quick Connect Coupling"@en ;
-                         edo:hasDiscipline edo:SubseaUmbilicalsEngineering ;
-                         edo:ifc_equivalentClass "IfcPipeFitting" ;
-                         edo:ifc_objectType "QuickConnectCoupling" .
+                         edo:hasDiscipline edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#QuickDisconnectionTool
@@ -7019,10 +6810,7 @@ edo:RearPoint rdf:type owl:Class ;
               skos:prefLabel "Ponto Traseiro"@pt-br ,
                              "Rear Point"@en ;
               edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ,
-                                edo:SubseaUmbilicalsEngineering ;
-              edo:ifc_equivalentClass "IfcDistributionPort" ;
-              edo:ifc_objectType "RearPoint" ;
-              edo:ifc_predefinedType "USERDEFINED" .
+                                edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#Recycling
@@ -7070,9 +6858,7 @@ edo:RigidJoint rdf:type owl:Class ;
                                "Temporary accessory for connection between the armour pot and the bend stiffener below it, in order to prevent excessive line bending during handling operations."@en ;
                skos:prefLabel "Junta Rígida"@pt-br ,
                               "Rigid Joint"@en ;
-               edo:hasDiscipline edo:SubseaUmbilicalsEngineering ;
-               edo:ifc_equivalentClass "IfcPipeFitting" ;
-               edo:ifc_objectType "RigidJoint" .
+               edo:hasDiscipline edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#RigidJumper
@@ -7107,10 +6893,7 @@ edo:RingGasket rdf:type owl:Class ;
                                 edo:RingGasketStandard ,
                                 edo:RingGasketType ;
                edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ,
-                                 edo:SubseaUmbilicalsEngineering ;
-               edo:ifc_equivalentClass "IfcMechanicalFastener" ;
-               edo:ifc_objectType "RingGasket" ;
-               edo:ifc_predefinedType "USERDEFINED" .
+                                 edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#RingGasketAttribute
@@ -7415,10 +7198,7 @@ edo:SacrificialSheath rdf:type owl:Class ;
                       skos:definition "A protective layer in subsea flexible pipes designed to absorb damage from external environmental factors, such as abrasion or impact, over time. The Sacrificial Sheath is intended to wear out or degrade under harsh conditions, thereby preserving the integrity of the underlying layers and prolonging the overall lifespan of the pipe. Made from durable materials, this sheath acts as the first line of defense, allowing for the controlled degradation of the pipe's outer surface without affecting its core structural or functional performance."@en ;
                       skos:prefLabel "Camada de Sacrifício"@pt-br ,
                                      "Sacrificial Sheath"@en ;
-                      edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ;
-                      edo:ifc_equivalentClass "IfcPipeSegment" ;
-                      edo:ifc_objectType "SacrificialSheath" ;
-                      edo:ifc_predefinedType "USERDEFINED" .
+                      edo:hasDiscipline edo:SubseaFlexiblePipesEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#SafeWorkingLoad
@@ -7527,10 +7307,7 @@ edo:Shackle rdf:type owl:Class ;
                              edo:ShackleOuterDiameter ,
                              edo:ShackleThickness ,
                              edo:WorkingLoadLimit ;
-            edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ;
-            edo:ifc_equivalentClass "IfcMechanicalFastener" ;
-            edo:ifc_objectType "Shackle" ;
-            edo:ifc_predefinedType "USERDEFINED" .
+            edo:hasDiscipline edo:SubseaFlexiblePipesEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#ShackleInnerDiameter
@@ -7954,10 +7731,7 @@ edo:StopperCollar rdf:type owl:Class ;
                   skos:prefLabel "Colar Batente"@pt-br ,
                                  "Stopper Collar"@en ;
                   edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ,
-                                    edo:SubseaUmbilicalsEngineering ;
-                  edo:ifc_equivalentClass "IfcPipeFitting" ;
-                  edo:ifc_objectType "StopperCollar" ;
-                  edo:ifc_predefinedType "USERDEFINED" .
+                                    edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#StorageAccessory
@@ -8344,10 +8118,7 @@ edo:SubseaConnectionPoint rdf:type owl:Class ;
                                           "Represents a connection point located in subsea environments, used to link subsea components."@en ;
                           skos:prefLabel "Ponto de Conexão Submarino"@pt-br ,
                                          "Subsea Connection Point"@en ;
-                          edo:hasDiscipline edo:SubseaUmbilicalsEngineering ;
-                          edo:ifc_equivalentClass "IfcPipeFitting" ;
-                          edo:ifc_objectType "SubseaConnectionPoint" ;
-                          edo:ifc_predefinedType "USERDEFINED" .
+                          edo:hasDiscipline edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#SubseaEngineering
@@ -8375,10 +8146,7 @@ edo:SubseaFlexiblePipesBsddDictionary rdf:type owl:Class ;
                                       dcterms:identifier "SubseaFlexiblePipesBsddDictionary" ;
                                       skos:definition "Bsdd Dictionary of Subsea Flexible Pipelines."@en ;
                                       skos:prefLabel "Dicionário Bsdd de Dutos Flexíveis Submarinos"@pt-br ,
-                                                     "Subsea Flexible Pipes Bsdd Dictionary"@en ;
-                                      edo:ifc_equivalentClass "IfcClassification" ;
-                                      edo:ifc_objectType "SubseaFlexiblePipesBsddDictionary" ;
-                                      edo:ifc_predefinedType "USERDEFINED" .
+                                                     "Subsea Flexible Pipes Bsdd Dictionary"@en .
 
 
 ###  https://w3id.org/energy-domain/edo#SubseaFlexiblePipesEngineering
@@ -8416,9 +8184,7 @@ edo:SubseaOilField rdf:type owl:Class ;
                    skos:definition "A subsea oil field refers to an underwater reservoir of hydrocarbons located beneath the seabed. It involves the extraction of oil and gas from deposits situated in deepwater or shallow marine environments. The subsea oil field includes a range of equipment and infrastructure, such as subsea wells, pipelines, and production systems, all designed to safely and efficiently extract and transport hydrocarbons to processing facilities on the surface or shore. The development and operation of subsea oil fields require specialized technology and expertise to manage the challenges of deepwater environments and ensure the successful production and handling of resources."@en ;
                    skos:prefLabel "Campo de Petróleo Submarino"@pt-br ,
                                   "Subsea Oil Field"@en ;
-                   edo:hasDiscipline edo:SubseaUmbilicalsEngineering ;
-                   edo:ifc_equivalentClass "IfcSite" ;
-                   edo:ifc_objectType "SubseaOilField" .
+                   edo:hasDiscipline edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#SubseaPipeline
@@ -8455,10 +8221,7 @@ edo:SubseaPipeline rdf:type owl:Class ;
                                     edo:SubProjectId ,
                                     edo:UnifilarDiagram ,
                                     edo:UnifilarDiagramRev ;
-                   edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ;
-                   edo:ifc_equivalentClass "IfcBuilding" ;
-                   edo:ifc_objectType "SubseaPipeline" ;
-                   edo:ifc_predefinedType "USERDEFINED" .
+                   edo:hasDiscipline edo:SubseaFlexiblePipesEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#SubseaRigidPipeSpec
@@ -8495,10 +8258,7 @@ edo:SubseaUmbilical rdf:type owl:Class ;
                     dcterms:identifier "SubseaUmbilical" ;
                     skos:definition "A Subsea Umbilical is a composite cable system installed on the seafloor that provides vital control, power, and communication connections between topside facilities and subsea production systems. Designed to transmit hydraulic fluids, electrical power, and data signals, the Subsea Umbilical enables the remote operation and monitoring of subsea equipment such as valves, manifolds, and control modules. Engineered for long-term reliability in extreme underwater environments, it is constructed using multiple layers of protective sheathing, armoring, and insulation materials to resist mechanical stress, corrosion, and hydrostatic pressure, ensuring safe and continuous operation throughout the field’s lifecycle."@en ;
                     skos:prefLabel "Subsea Umbilical"@en ,
-                                   "Umbilical Submarino"@pt-br ;
-                    edo:ifc_equivalentClass "IfcBuilding" ;
-                    edo:ifc_objectType "SubseaUmbilical" ;
-                    edo:ifc_predefinedType "USERDEFINED" .
+                                   "Umbilical Submarino"@pt-br .
 
 
 ###  https://w3id.org/energy-domain/edo#SubseaUmbilicalBsddDictionary
@@ -8507,10 +8267,7 @@ edo:SubseaUmbilicalBsddDictionary rdf:type owl:Class ;
                                   dcterms:identifier "SubseaUmbilicalBsddDictionary" ;
                                   skos:definition "Bsdd Dictionary of Subsea Flexible Pipelines."@en ;
                                   skos:prefLabel "Dicionário Bsdd de Umbilicais Submarinos"@pt-br ,
-                                                 "Subsea Umbilical Bsdd Dictionary"@en ;
-                                  edo:ifc_equivalentClass "IfcClassification" ;
-                                  edo:ifc_objectType "SubseaUmbilicalBsddDictionary" ;
-                                  edo:ifc_predefinedType "USERDEFINED" .
+                                                 "Subsea Umbilical Bsdd Dictionary"@en .
 
 
 ###  https://w3id.org/energy-domain/edo#SubseaUmbilicalsEngineering
@@ -8530,10 +8287,7 @@ edo:SubseaWell rdf:type owl:Class ;
                skos:definition "A Subsea Well is a well located on the seafloor from which hydrocarbons such as oil and gas are extracted. Positioned underwater, it typically involves a wellhead assembly that includes valves, controls, and monitoring equipment to manage the extraction process. The Subsea Well is connected to surface facilities via pipelines and umbilicals, which facilitate the transport of extracted resources and the provision of necessary services like power and communication. Designed to operate under high pressure and harsh marine conditions, the subsea well is a critical component of offshore oil and gas production systems."@en ;
                skos:prefLabel "Poço"@pt-br ,
                               "Subsea Well"@en ;
-               edo:hasDiscipline edo:SubseaUmbilicalsEngineering ;
-               edo:ifc_equivalentClass "IfcBuilding" ;
-               edo:ifc_objectType "SubseaWell" ;
-               edo:ifc_predefinedType "USERDEFINED" .
+               edo:hasDiscipline edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#SuctionPile
@@ -8681,10 +8435,7 @@ edo:TensileArmour rdf:type owl:Class ;
                   edo:hasAttribute edo:TensileWireLayAngle ,
                                    edo:TensileWiresCount ;
                   edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ,
-                                    edo:SubseaUmbilicalsEngineering ;
-                  edo:ifc_equivalentClass "IfcPipeSegment" ;
-                  edo:ifc_objectType "TensileArmour" ;
-                  edo:ifc_predefinedType "USERDEFINED" .
+                                    edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#TensileArmourAnnulusCondition
@@ -8918,9 +8669,7 @@ edo:ThreadedFitting rdf:type owl:Class ;
                     skos:definition "A Threaded Fitting is a connector used to join fluid or gas lines through screw threads, providing a secure and leak-tight connection that can be easily assembled or disassembled during installation or maintenance."@en ,
                                     "Um Engate Rosqueado é um conector usado para unir linhas de fluido ou gás por meio de roscas, garantindo uma conexão segura e vedada que pode ser facilmente montada ou desmontada durante a instalação ou manutenção."@pt-br ;
                     skos:prefLabel "Engate Rosqueado"@pt-br ,
-                                   "Threaded Fitting"@en ;
-                    edo:ifc_equivalentClass "IfcPipeFitting" ;
-                    edo:ifc_objectType "ThreadedFitting" .
+                                   "Threaded Fitting"@en .
 
 
 ###  https://w3id.org/energy-domain/edo#ThreadsPerInch
@@ -9007,9 +8756,7 @@ edo:TopBendStiffener rdf:type owl:Class ;
                      skos:prefLabel "Enrijecedor de Curvatura de Topo"@pt-br ,
                                     "Top Bend Stiffener"@en ;
                      edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ,
-                                       edo:SubseaUmbilicalsEngineering ;
-                     edo:ifc_equivalentClass "IfcPipeFitting" ;
-                     edo:ifc_objectType "TopBendStiffener" .
+                                       edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#TopsideConnectionPoint
@@ -9022,10 +8769,7 @@ edo:TopsideConnectionPoint rdf:type owl:Class ;
                            skos:prefLabel "Ponto de Conexão em Topside"@pt-br ,
                                           "Topside Connection Point"@en ;
                            edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ,
-                                             edo:SubseaUmbilicalsEngineering ;
-                           edo:ifc_equivalentClass "IfcPipeFitting" ;
-                           edo:ifc_objectType "TopsideConnectionPoint" ;
-                           edo:ifc_predefinedType "USERDEFINED" .
+                                             edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#TopsideEquipment
@@ -9159,10 +8903,7 @@ edo:UNSBoltSet rdf:type owl:Class ;
                                 edo:ThreadStandard ,
                                 edo:ThreadsPerInch ;
                edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ,
-                                 edo:SubseaUmbilicalsEngineering ;
-               edo:ifc_equivalentClass "IfcMechanicalFastener" ;
-               edo:ifc_objectType "UNSBoltSet" ;
-               edo:ifc_predefinedType "USERDEFINED" .
+                                 edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#UNSStudSet
@@ -9177,10 +8918,7 @@ edo:UNSStudSet rdf:type owl:Class ;
                                 edo:ThreadStandard ,
                                 edo:ThreadsPerInch ;
                edo:hasDiscipline edo:SubseaFlexiblePipesEngineering ,
-                                 edo:SubseaUmbilicalsEngineering ;
-               edo:ifc_equivalentClass "IfcMechanicalFastener" ;
-               edo:ifc_objectType "UNSStudSet" ;
-               edo:ifc_predefinedType "USERDEFINED" .
+                                 edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#UTA
@@ -9254,10 +8992,7 @@ edo:UmbilicalBundle rdf:type owl:Class ;
                     skos:definition "O Conjunto de Umbilical é o agrupamento integrado dos elementos funcionais que compõem um umbilical submarino, projetado para transmitir potência hidráulica, energia elétrica e sinais de dados entre os sistemas de superfície e submarinos. Normalmente é formado por vários tubos, cabos e fibras reunidos em um único feixe, protegidos por camadas externas de revestimento e armadura. Projetado para garantir confiabilidade e eficiência em condições de águas profundas, o conjunto de umbilical fornece os canais essenciais de controle e comunicação que permitem a operação segura e contínua dos equipamentos submarinos de produção."@pt-br ,
                                     "The Umbilical Bundle is the integrated assembly of functional elements within a subsea umbilical, designed to transmit hydraulic power, electrical energy, and data signals between surface and subsea systems. It typically consists of multiple tubes, cables, and fibers bundled together and protected by external sheathing and armoring. Engineered to ensure reliability and efficiency under deepwater conditions, the umbilical bundle provides essential control and communication pathways that enable the safe and continuous operation of subsea production equipment."@en ;
                     skos:prefLabel "Conjunto de Umbilical"@pt-br ,
-                                   "Umbilical Bundle"@en ;
-                    edo:ifc_equivalentClass "IfcPipeSegmentType" ;
-                    edo:ifc_objectType "UmbilicalBundle" ;
-                    edo:ifc_predefinedType "USERDEFINED" .
+                                   "Umbilical Bundle"@en .
 
 
 ###  https://w3id.org/energy-domain/edo#UmbilicalComponent
@@ -9291,9 +9026,7 @@ edo:UmbilicalPullingHead rdf:type owl:Class ;
                                          "Tubular pulling head (with space for pigtails), used for umbilicals."@en ;
                          skos:prefLabel "Cabeça de Tração para Umbilical"@pt-br ,
                                         "Umbilical Pulling Head"@en ;
-                         edo:hasDiscipline edo:SubseaUmbilicalsEngineering ;
-                         edo:ifc_equivalentClass "IfcPipeFitting" ;
-                         edo:ifc_objectType "UmbilicalPullingHead" .
+                         edo:hasDiscipline edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#UmbilicalSegment
@@ -9303,10 +9036,7 @@ edo:UmbilicalSegment rdf:type owl:Class ;
                      dcterms:identifier "UmbilicalSegment" ;
                      skos:prefLabel "Tramo de Umbilical"@pt-br ,
                                     "Umbilical Segment"@en ;
-                     edo:hasDiscipline edo:SubseaUmbilicalsEngineering ;
-                     edo:ifc_equivalentClass "UmbilicalSegment" ;
-                     edo:ifc_objectType "IfcPipeSegment" ;
-                     edo:ifc_predefinedType "USERDEFINED" .
+                     edo:hasDiscipline edo:SubseaUmbilicalsEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#UmbilicalSpan
@@ -9325,10 +9055,7 @@ edo:UmbilicalStructure rdf:type owl:Class ;
                        skos:definition "A Estrutura do Umbilical é a configuração em camadas, a especificação, de um umbilical submarino, projetada para fornecer proteção mecânica, flexibilidade e estabilidade em ambientes subaquáticos. Ela é composta por múltiplas camadas e componentes, como tubos, cabos, materiais de enchimento e armaduras, cada um desempenhando uma função específica, incluindo transmissão de energia, comunicação de sinais ou reforço estrutural. Essa estrutura é projetada para suportar as condições de águas profundas, como alta pressão externa e cargas dinâmicas, garantindo a integridade e o desempenho confiável do umbilical ao longo de sua vida útil."@pt-br ,
                                        "The Umbilical Structure is the layered configuration, the specification, of a subsea umbilical, designed to provide mechanical protection, flexibility, and stability in underwater environments. It consists of multiple layers and components, such as tubes, cables, fillers, and armoring, each serving a specific function, including power transmission, signal communication, or structural reinforcement. This structure is engineered to withstand deepwater conditions, such as high external pressure and dynamic loads, ensuring the umbilical’s integrity and reliable performance throughout its service life.."@en ;
                        skos:prefLabel "Estrutura do Umbilical"@pt-br ,
-                                      "Umbilical Structure"@en ;
-                       edo:ifc_equivalentClass "IfcPipeSegmentType" ;
-                       edo:ifc_objectType "UmbilicalStructure" ;
-                       edo:ifc_predefinedType "USERDEFINED" .
+                                      "Umbilical Structure"@en .
 
 
 ###  https://w3id.org/energy-domain/edo#UnifilarDiagram
@@ -9505,10 +9232,7 @@ edo:WeldedConnection rdf:type owl:Class ;
                      skos:definition "Representa uma conexão de porta de fluido realizada por soldagem, formando uma junta permanente. Também define a localização lógica em um modelo onde os componentes de distribuição de fluido são unidos, garantindo o alinhamento e a funcionalidade corretos durante a montagem e a operação. Essa conexão lógica é essencial para definir a interface entre elementos de distribuição de fluido em sistemas submarinos ou outros sistemas de manuseio de fluido."@pt-br ,
                                      "Represents a fluid port connection achieved by welding, forming a permanent joint. It also defines the logical location within a model where fluid distribution components are joined, ensuring proper alignment and functionality during assembly and operation. This logical connection is essential for defining the interface between fluid distribution elements in subsea systems or other fluid handling applications."@en ;
                      skos:prefLabel "Conexão Soldada"@pt-br ,
-                                    "Welded Connection"@en ;
-                     edo:ifc_equivalentClass "IfcDistributionPort" ;
-                     edo:ifc_objectType "WeldedConnection" ;
-                     edo:ifc_predefinedType "USERDEFINED" .
+                                    "Welded Connection"@en .
 
 
 ###  https://w3id.org/energy-domain/edo#Wellhead
@@ -9527,9 +9251,7 @@ edo:WetChristmasTree rdf:type owl:Class ;
                                      "Piece of equipment containing valves and instrumentation allowing direct control of flow from/to a well"@en ;
                      skos:prefLabel "Wet Christmas tree"@en ,
                                     "Árvore de Natal molhada"@pt-br ;
-                     edo:hasDiscipline edo:WetChristmasTreesEngineering ;
-                     edo:ifc_objectType "WetChristmasTree" ;
-                     edo:ifc_predefinedType "USERDEFINED" .
+                     edo:hasDiscipline edo:WetChristmasTreesEngineering .
 
 
 ###  https://w3id.org/energy-domain/edo#WetChristmasTreesEngineering
diff --git a/mappings/ifc/edo-ifc.ttl b/mappings/ifc/edo-ifc.ttl
index 1567e6d..a7fd531 100644
--- a/mappings/ifc/edo-ifc.ttl
+++ b/mappings/ifc/edo-ifc.ttl
@@ -462,6 +462,198 @@ edo-ifc:IfcPipeSegment
 
     dcterms:identifier "IfcPipeSegment" .
 
+
+######################################################################
+# 8A) Controlled IFC entities migrated from EDO core
+######################################################################
+
+edo-ifc:IfcActor
+    rdf:type owl:NamedIndividual ;
+    rdf:type edo-ifc:IFCEntity ;
+
+    rdfs:label "IfcActor"@en ;
+    edo-ifc:ifcEntityName "IfcActor" ;
+
+    rdfs:seeAlso <https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/lexical/IfcActor.htm> ;
+
+    dcterms:identifier "IfcActor" .
+
+edo-ifc:IfcBuilding
+    rdf:type owl:NamedIndividual ;
+    rdf:type edo-ifc:IFCEntity ;
+
+    rdfs:label "IfcBuilding"@en ;
+    edo-ifc:ifcEntityName "IfcBuilding" ;
+
+    rdfs:seeAlso <https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/lexical/IfcBuilding.htm> ;
+
+    dcterms:identifier "IfcBuilding" .
+
+edo-ifc:IfcCableFitting
+    rdf:type owl:NamedIndividual ;
+    rdf:type edo-ifc:IFCEntity ;
+
+    rdfs:label "IfcCableFitting"@en ;
+    edo-ifc:ifcEntityName "IfcCableFitting" ;
+
+    rdfs:seeAlso <https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/lexical/IfcCableFitting.htm> ;
+
+    dcterms:identifier "IfcCableFitting" .
+
+edo-ifc:IfcCableSegment
+    rdf:type owl:NamedIndividual ;
+    rdf:type edo-ifc:IFCEntity ;
+
+    rdfs:label "IfcCableSegment"@en ;
+    edo-ifc:ifcEntityName "IfcCableSegment" ;
+
+    rdfs:seeAlso <https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/lexical/IfcCableSegment.htm> ;
+
+    dcterms:identifier "IfcCableSegment" .
+
+edo-ifc:IfcClassification
+    rdf:type owl:NamedIndividual ;
+    rdf:type edo-ifc:IFCEntity ;
+
+    rdfs:label "IfcClassification"@en ;
+    edo-ifc:ifcEntityName "IfcClassification" ;
+
+    rdfs:seeAlso <https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/lexical/IfcClassification.htm> ;
+
+    dcterms:identifier "IfcClassification" .
+
+edo-ifc:IfcClassificationReference
+    rdf:type owl:NamedIndividual ;
+    rdf:type edo-ifc:IFCEntity ;
+
+    rdfs:label "IfcClassificationReference"@en ;
+    edo-ifc:ifcEntityName "IfcClassificationReference" ;
+
+    rdfs:seeAlso <https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/lexical/IfcClassificationReference.htm> ;
+
+    dcterms:identifier "IfcClassificationReference" .
+
+edo-ifc:IfcDocumentInformation
+    rdf:type owl:NamedIndividual ;
+    rdf:type edo-ifc:IFCEntity ;
+
+    rdfs:label "IfcDocumentInformation"@en ;
+    edo-ifc:ifcEntityName "IfcDocumentInformation" ;
+
+    rdfs:seeAlso <https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/lexical/IfcDocumentInformation.htm> ;
+
+    dcterms:identifier "IfcDocumentInformation" .
+
+edo-ifc:IfcGroup
+    rdf:type owl:NamedIndividual ;
+    rdf:type edo-ifc:IFCEntity ;
+
+    rdfs:label "IfcGroup"@en ;
+    edo-ifc:ifcEntityName "IfcGroup" ;
+
+    rdfs:seeAlso <https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/lexical/IfcGroup.htm> ;
+
+    dcterms:identifier "IfcGroup" .
+
+edo-ifc:IfcMaterial
+    rdf:type owl:NamedIndividual ;
+    rdf:type edo-ifc:IFCEntity ;
+
+    rdfs:label "IfcMaterial"@en ;
+    edo-ifc:ifcEntityName "IfcMaterial" ;
+
+    rdfs:seeAlso <https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/lexical/IfcMaterial.htm> ;
+
+    dcterms:identifier "IfcMaterial" .
+
+edo-ifc:IfcMechanicalFastener
+    rdf:type owl:NamedIndividual ;
+    rdf:type edo-ifc:IFCEntity ;
+
+    rdfs:label "IfcMechanicalFastener"@en ;
+    edo-ifc:ifcEntityName "IfcMechanicalFastener" ;
+
+    rdfs:seeAlso <https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/lexical/IfcMechanicalFastener.htm> ;
+
+    dcterms:identifier "IfcMechanicalFastener" .
+
+edo-ifc:IfcPerson
+    rdf:type owl:NamedIndividual ;
+    rdf:type edo-ifc:IFCEntity ;
+
+    rdfs:label "IfcPerson"@en ;
+    edo-ifc:ifcEntityName "IfcPerson" ;
+
+    rdfs:seeAlso <https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/lexical/IfcPerson.htm> ;
+
+    dcterms:identifier "IfcPerson" .
+
+edo-ifc:IfcPipeFitting
+    rdf:type owl:NamedIndividual ;
+    rdf:type edo-ifc:IFCEntity ;
+
+    rdfs:label "IfcPipeFitting"@en ;
+    edo-ifc:ifcEntityName "IfcPipeFitting" ;
+
+    rdfs:seeAlso <https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/lexical/IfcPipeFitting.htm> ;
+
+    dcterms:identifier "IfcPipeFitting" .
+
+edo-ifc:IfcPipeSegmentType
+    rdf:type owl:NamedIndividual ;
+    rdf:type edo-ifc:IFCEntity ;
+
+    rdfs:label "IfcPipeSegmentType"@en ;
+    edo-ifc:ifcEntityName "IfcPipeSegmentType" ;
+
+    rdfs:seeAlso <https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/lexical/IfcPipeSegmentType.htm> ;
+
+    dcterms:identifier "IfcPipeSegmentType" .
+
+edo-ifc:IfcProject
+    rdf:type owl:NamedIndividual ;
+    rdf:type edo-ifc:IFCEntity ;
+
+    rdfs:label "IfcProject"@en ;
+    edo-ifc:ifcEntityName "IfcProject" ;
+
+    rdfs:seeAlso <https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/lexical/IfcProject.htm> ;
+
+    dcterms:identifier "IfcProject" .
+
+edo-ifc:IfcProtectiveDevice
+    rdf:type owl:NamedIndividual ;
+    rdf:type edo-ifc:IFCEntity ;
+
+    rdfs:label "IfcProtectiveDevice"@en ;
+    edo-ifc:ifcEntityName "IfcProtectiveDevice" ;
+
+    rdfs:seeAlso <https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/lexical/IfcProtectiveDevice.htm> ;
+
+    dcterms:identifier "IfcProtectiveDevice" .
+
+edo-ifc:IfcSite
+    rdf:type owl:NamedIndividual ;
+    rdf:type edo-ifc:IFCEntity ;
+
+    rdfs:label "IfcSite"@en ;
+    edo-ifc:ifcEntityName "IfcSite" ;
+
+    rdfs:seeAlso <https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/lexical/IfcSite.htm> ;
+
+    dcterms:identifier "IfcSite" .
+
+edo-ifc:IfcTask
+    rdf:type owl:NamedIndividual ;
+    rdf:type edo-ifc:IFCEntity ;
+
+    rdfs:label "IfcTask"@en ;
+    edo-ifc:ifcEntityName "IfcTask" ;
+
+    rdfs:seeAlso <https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/lexical/IfcTask.htm> ;
+
+    dcterms:identifier "IfcTask" .
+
 ######################################################################
 # 9) Controlled IFC attribute roles
 ######################################################################
@@ -637,6 +829,449 @@ edo-ifc:AnyRealizingElementRequirement
 
     dcterms:identifier "AnyRealizingElementRequirement" .
 
+
+######################################################################
+# 11A) Direct IFC mappings for EDO classes migrated from core
+######################################################################
+
+edo:AbandonmentCap
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcCableFitting ;
+    edo-ifc:ifc_objectType "AbandonmentCap" .
+
+edo:AbrasionProtector
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcProtectiveDevice ;
+    edo-ifc:ifc_objectType "AbrasionProtector" .
+
+edo:AnchoringCollar
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeFitting ;
+    edo-ifc:ifc_objectType "AnchoringCollar" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:AnodeCollarSet
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeFitting ;
+    edo-ifc:ifc_objectType "AnodeCollarSet" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:AntiCollapseSheath
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeSegment ;
+    edo-ifc:ifc_objectType "AntiCollapseSheath" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:AntiWearSheath
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeSegment ;
+    edo-ifc:ifc_objectType "AntiWearSheath" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:ArmorPot
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeFitting ;
+    edo-ifc:ifc_objectType "ArmorPot" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:BendRestrictor
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeFitting ;
+    edo-ifc:ifc_objectType "BendRestrictor" .
+
+edo:BuoyancyModule
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeFitting ;
+    edo-ifc:ifc_objectType "BuoyancyModule" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:CommissioningActivity
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcTask ;
+    edo-ifc:ifc_objectType "Activity" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:CommissioningContract
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcGroup ;
+    edo-ifc:ifc_objectType "Contract" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:CommissioningDigitalProcessStep
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcTask ;
+    edo-ifc:ifc_objectType "CommissioningDigitalProcessStep" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:CommissioningEvidence
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcDocumentInformation ;
+    edo-ifc:ifc_objectType "CommissioningEvidence" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:CommissioningIssue
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcClassification ;
+    edo-ifc:ifc_objectType "Pendency_Classification" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:CommissioningItemCheck
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcClassificationReference ;
+    edo-ifc:ifc_objectType "FVI" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:CommissioningLoopCheck
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcClassificationReference ;
+    edo-ifc:ifc_objectType "FVM" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:CommissioningPerson
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPerson ;
+    edo-ifc:ifc_objectType "CommissioningPerson" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:CommissioningPreservationOrder
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcClassificationReference ;
+    edo-ifc:ifc_objectType "OP" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:CommissioningProgram
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcGroup ;
+    edo-ifc:ifc_objectType "Program" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:CommissioningProject
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcProject ;
+    edo-ifc:ifc_objectType "Project" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:CommissioningResponsibleActor
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcActor ;
+    edo-ifc:ifc_objectType "CommissioningResponsibleActor" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:CommissioningResponsibleGroup
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcGroup ;
+    edo-ifc:ifc_objectType "CommissioningResponsibleGroup" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:CommissioningTask
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcTask ;
+    edo-ifc:ifc_objectType "Task" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:CompositeMaterial
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcMaterial ;
+    edo-ifc:ifc_objectType "CompositeMaterial" .
+
+edo:DeadweightCollar
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeFitting ;
+    edo-ifc:ifc_objectType "DeadweightCollar" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:DimensionsDrawing
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcDocumentInformation ;
+    edo-ifc:ifc_objectType "DimensionsDrawing" .
+
+edo:ElectricalCable
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcCableSegment ;
+    edo-ifc:ifc_objectType "ElectricalCable" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:ElectricalJumperConnector
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcCableFitting ;
+    edo-ifc:ifc_objectType "ElectricalJumperConnector" .
+
+edo:ElectricalPowerJumperConnector
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcCableFitting ;
+    edo-ifc:ifc_objectType "ElectricalPowerJumperConnector" .
+
+edo:EndFitting
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeFitting ;
+    edo-ifc:ifc_objectType "EndFitting" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:FabricTape
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeSegment ;
+    edo-ifc:ifc_objectType "FabricTape" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:FlangeAdapter
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeFitting ;
+    edo-ifc:ifc_objectType "FlangeAdapter" .
+
+edo:FlangeConnection
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcDistributionPort ;
+    edo-ifc:ifc_objectType "FlangeConnection" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:FlexiblePipeSegment
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeSegment ;
+    edo-ifc:ifc_objectType "FlexiblePipeSegment" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:FlexiblePipeStructure
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeSegmentType ;
+    edo-ifc:ifc_objectType "FlexiblePipeStructure" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:FloatingProductionUnit
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcBuilding ;
+    edo-ifc:ifc_objectType "FloatingProductionUnit" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:GroovePoint
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcDistributionPort ;
+    edo-ifc:ifc_objectType "GroovePoint" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:HCRHose
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeSegment ;
+    edo-ifc:ifc_objectType "HCRHose" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:HandlingCollar
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeFitting ;
+    edo-ifc:ifc_objectType "HandlingCollar" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:HangOffCollar
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeFitting ;
+    edo-ifc:ifc_objectType "HangOffCollar" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:HighStrengthTape
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeSegment ;
+    edo-ifc:ifc_objectType "HighStrengthTape" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:HoldingBandage
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeSegment ;
+    edo-ifc:ifc_objectType "HoldingBandage" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:HotStab
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeFitting ;
+    edo-ifc:ifc_objectType "HotStab" .
+
+edo:HotStabReceptacle
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeFitting ;
+    edo-ifc:ifc_objectType "HotStabReceptacle" .
+
+edo:IfcInstanciableElement
+    edo-ifc:ifc_equivalentClass edo-ifc:- ;
+    edo-ifc:ifc_objectType "-" ;
+    edo-ifc:ifc_predefinedType "-" .
+
+edo:InnerTube
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeSegment ;
+    edo-ifc:ifc_objectType "InnerTube" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:InsulationTape
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeSegment ;
+    edo-ifc:ifc_objectType "InsulationTape" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:IntegratedPipe
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeSegment ;
+    edo-ifc:ifc_objectType "IntegratedPipe" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:InterlockedCarcass
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeSegment ;
+    edo-ifc:ifc_objectType "InterlockedCarcass" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:IntermediateBendStiffener
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeFitting ;
+    edo-ifc:ifc_objectType "IntermediateBendStiffener" .
+
+edo:IntermediateSheath
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeSegment ;
+    edo-ifc:ifc_objectType "IntermediateSheath" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:InternalPressureSheath
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeSegment ;
+    edo-ifc:ifc_objectType "InternalPressureSheath" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:LowResistanceTape
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeSegment ;
+    edo-ifc:ifc_objectType "LowResistanceTape" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:MetallicMaterial
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcMaterial ;
+    edo-ifc:ifc_objectType "MetallicMaterial" .
+
+edo:MetallicStrandSet
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcCableSegment ;
+    edo-ifc:ifc_objectType "MetallicStrandSet" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:MetallicTubing
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeSegment ;
+    edo-ifc:ifc_objectType "MetallicTubing" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:MetricBoltSet
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcMechanicalFastener ;
+    edo-ifc:ifc_objectType "MetricBoltSet" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:MetricStudSet
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcMechanicalFastener ;
+    edo-ifc:ifc_objectType "MetricStudSet" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:OpticalFiberCable
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcCableSegment ;
+    edo-ifc:ifc_objectType "OpticalFiberCable" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:PipePullingHead
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeFitting ;
+    edo-ifc:ifc_objectType "PipePullingHead" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:PlasticHose
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeSegment ;
+    edo-ifc:ifc_objectType "PlasticHose" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:PolymericMaterial
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcMaterial ;
+    edo-ifc:ifc_objectType "PolymericMaterial" .
+
+edo:PressureArmour
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeSegment ;
+    edo-ifc:ifc_objectType "PressureArmour" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:Project
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcProject .
+
+edo:ProtectiveSheath
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeSegment ;
+    edo-ifc:ifc_objectType "ProtectiveSheath" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:PullInCollar
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeFitting ;
+    edo-ifc:ifc_objectType "PullInCollar" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:QuickConnectCoupling
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeFitting ;
+    edo-ifc:ifc_objectType "QuickConnectCoupling" .
+
+edo:RearPoint
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcDistributionPort ;
+    edo-ifc:ifc_objectType "RearPoint" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:RigidJoint
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeFitting ;
+    edo-ifc:ifc_objectType "RigidJoint" .
+
+edo:RingGasket
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcMechanicalFastener ;
+    edo-ifc:ifc_objectType "RingGasket" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:SacrificialSheath
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeSegment ;
+    edo-ifc:ifc_objectType "SacrificialSheath" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:Shackle
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcMechanicalFastener ;
+    edo-ifc:ifc_objectType "Shackle" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:StopperCollar
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeFitting ;
+    edo-ifc:ifc_objectType "StopperCollar" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:SubseaConnectionPoint
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeFitting ;
+    edo-ifc:ifc_objectType "SubseaConnectionPoint" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:SubseaFlexiblePipesBsddDictionary
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcClassification ;
+    edo-ifc:ifc_objectType "SubseaFlexiblePipesBsddDictionary" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:SubseaOilField
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcSite ;
+    edo-ifc:ifc_objectType "SubseaOilField" .
+
+edo:SubseaPipeline
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcBuilding ;
+    edo-ifc:ifc_objectType "SubseaPipeline" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:SubseaUmbilical
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcBuilding ;
+    edo-ifc:ifc_objectType "SubseaUmbilical" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:SubseaUmbilicalBsddDictionary
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcClassification ;
+    edo-ifc:ifc_objectType "SubseaUmbilicalBsddDictionary" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:SubseaWell
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcBuilding ;
+    edo-ifc:ifc_objectType "SubseaWell" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:TensileArmour
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeSegment ;
+    edo-ifc:ifc_objectType "TensileArmour" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:ThreadedFitting
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeFitting ;
+    edo-ifc:ifc_objectType "ThreadedFitting" .
+
+edo:TopBendStiffener
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeFitting ;
+    edo-ifc:ifc_objectType "TopBendStiffener" .
+
+edo:TopsideConnectionPoint
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeFitting ;
+    edo-ifc:ifc_objectType "TopsideConnectionPoint" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:UNSBoltSet
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcMechanicalFastener ;
+    edo-ifc:ifc_objectType "UNSBoltSet" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:UNSStudSet
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcMechanicalFastener ;
+    edo-ifc:ifc_objectType "UNSStudSet" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:UmbilicalBundle
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeSegmentType ;
+    edo-ifc:ifc_objectType "UmbilicalBundle" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:UmbilicalPullingHead
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeFitting ;
+    edo-ifc:ifc_objectType "UmbilicalPullingHead" .
+
+edo:UmbilicalSegment
+    edo-ifc:ifc_equivalentClass edo-ifc:UmbilicalSegment ;
+    edo-ifc:ifc_objectType "IfcPipeSegment" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:UmbilicalStructure
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcPipeSegmentType ;
+    edo-ifc:ifc_objectType "UmbilicalStructure" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:WeldedConnection
+    edo-ifc:ifc_equivalentClass edo-ifc:IfcDistributionPort ;
+    edo-ifc:ifc_objectType "WeldedConnection" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
+edo:WetChristmasTree
+    edo-ifc:ifc_objectType "WetChristmasTree" ;
+    edo-ifc:ifc_predefinedType "USERDEFINED" .
+
 ######################################################################
 # 11) Direct IFC mappings for EDO relationships
 ######################################################################
