diff --git a/mappings/ifc/edo-ifc.ttl b/mappings/ifc/edo-ifc.ttl
index 30ed4d5..d72f867 100644
--- a/mappings/ifc/edo-ifc.ttl
+++ b/mappings/ifc/edo-ifc.ttl
@@ -386,6 +386,30 @@ edo-ifc:IfcRelAssignsToGroup
     dcterms:identifier "IfcRelAssignsToGroup" .
 
 
+edo-ifc:IfcRelAssignsToControl
+    rdf:type owl:NamedIndividual ;
+    rdf:type edo-ifc:IFCRelationshipEntity ;
+
+    rdfs:label "IfcRelAssignsToControl"@en ;
+    edo-ifc:ifcEntityName "IfcRelAssignsToControl" ;
+
+    rdfs:seeAlso <https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/lexical/IfcRelAssignsToControl.htm> ;
+
+    dcterms:identifier "IfcRelAssignsToControl" .
+
+
+edo-ifc:IfcRelContainedInSpatialStructure
+    rdf:type owl:NamedIndividual ;
+    rdf:type edo-ifc:IFCRelationshipEntity ;
+
+    rdfs:label "IfcRelContainedInSpatialStructure"@en ;
+    edo-ifc:ifcEntityName "IfcRelContainedInSpatialStructure" ;
+
+    rdfs:seeAlso <https://ifc43-docs.standards.buildingsmart.org/IFC/RELEASE/IFC4x3/HTML/lexical/IfcRelContainedInSpatialStructure.htm> ;
+
+    dcterms:identifier "IfcRelContainedInSpatialStructure" .
+
+
 edo-ifc:IfcRelAssignsToProduct
     rdf:type owl:NamedIndividual ;
     rdf:type edo-ifc:IFCRelationshipEntity ;
@@ -825,6 +849,26 @@ edo-ifc:RelatingGroup
     dcterms:identifier "RelatingGroup" .
 
 
+edo-ifc:RelatingControl
+    rdf:type owl:NamedIndividual ;
+    rdf:type edo-ifc:IFCAttributeRole ;
+
+    rdfs:label "RelatingControl"@en ;
+    edo-ifc:ifcAttributeName "RelatingControl" ;
+
+    dcterms:identifier "RelatingControl" .
+
+
+edo-ifc:RelatingStructure
+    rdf:type owl:NamedIndividual ;
+    rdf:type edo-ifc:IFCAttributeRole ;
+
+    rdfs:label "RelatingStructure"@en ;
+    edo-ifc:ifcAttributeName "RelatingStructure" ;
+
+    dcterms:identifier "RelatingStructure" .
+
+
 edo-ifc:RelatingProduct
     rdf:type owl:NamedIndividual ;
     rdf:type edo-ifc:IFCAttributeRole ;
