diff --git a/core/edo.ttl b/core/edo.ttl
index 9bdb70f..a443da0 100644
--- a/core/edo.ttl
+++ b/core/edo.ttl
@@ -1397,7 +1397,8 @@ edo:BooleanValue rdf:type owl:Class ;
 
 ###  https://w3id.org/energy-domain/edo#BoreDiameter
 edo:BoreDiameter rdf:type owl:Class ;
-                 rdfs:subClassOf edo:DomainAttribute ;
+                 rdfs:subClassOf edo:DomainAttribute ,
+                 edo:Diameter ;
                  dcterms:accessRights "PUBLIC" ;
                  dcterms:identifier "BoreDiameter" ;
                  skos:definition "Internal diameter that defines the size of the passage through which fluids or gases can flow. This dimension is essential in industrial piping, as it determines flow capacity and impacts pressure and velocity within the system."@en ;
@@ -1406,7 +1407,11 @@ edo:BoreDiameter rdf:type owl:Class ;
                  edo:hasAttributeScope edo:TypeLevelAttribute ;
                  edo:hasLifecycleCreationPhase edo:DetailedDesign ;
                  edo:hasTypedValue edo:FloatValue ;
-                 edo:hasValueCardinality edo:SingleValue .
+                 edo:hasValueCardinality edo:SingleValue ;
+                 edo:specializesAttribute edo:Diameter ;
+                 edo:hasUnit unit:M ;
+                 edo:lifecycleRole edo:DesignRole ;
+                 edo:attributeNature edo:DimensionalAttribute .
 
 
 ###  https://w3id.org/energy-domain/edo#BsddDataDictionary
@@ -1579,7 +1584,8 @@ edo:CivilEngineering rdf:type owl:Class ;
 
 ###  https://w3id.org/energy-domain/edo#ClampInternalDiameter
 edo:ClampInternalDiameter rdf:type owl:Class ;
-                          rdfs:subClassOf edo:DomainAttribute ;
+                          rdfs:subClassOf edo:DomainAttribute ,
+                          edo:Diameter ;
                           dcterms:accessRights "PUBLIC" ;
                           dcterms:identifier "ClampInternalDiameter" ;
                           skos:definition "Is the clamp internal diameter"@en ;
@@ -1588,12 +1594,17 @@ edo:ClampInternalDiameter rdf:type owl:Class ;
                           edo:hasAttributeScope edo:TypeLevelAttribute ;
                           edo:hasLifecycleCreationPhase edo:DetailedDesign ;
                           edo:hasTypedValue edo:FloatValue ;
-                          edo:hasValueCardinality edo:SingleValue .
+                          edo:hasValueCardinality edo:SingleValue ;
+                          edo:specializesAttribute edo:Diameter ;
+                          edo:hasUnit unit:M ;
+                          edo:lifecycleRole edo:DesignRole ;
+                          edo:attributeNature edo:DimensionalAttribute .
 
 
 ###  https://w3id.org/energy-domain/edo#CollarInternalDiameter
 edo:CollarInternalDiameter rdf:type owl:Class ;
-                           rdfs:subClassOf edo:DomainAttribute ;
+                           rdfs:subClassOf edo:DomainAttribute ,
+                           edo:Diameter ;
                            dcterms:accessRights "PUBLIC" ;
                            dcterms:identifier "CollarInternalDiameter" ;
                            skos:definition "Inner dimension of the collar, to ensure proper fitting and secure attachment."@en ;
@@ -1602,7 +1613,11 @@ edo:CollarInternalDiameter rdf:type owl:Class ;
                            edo:hasAttributeScope edo:TypeLevelAttribute ;
                            edo:hasLifecycleCreationPhase edo:DetailedDesign ;
                            edo:hasTypedValue edo:FloatValue ;
-                           edo:hasValueCardinality edo:SingleValue .
+                           edo:hasValueCardinality edo:SingleValue ;
+                           edo:specializesAttribute edo:Diameter ;
+                           edo:hasUnit unit:M ;
+                           edo:lifecycleRole edo:DesignRole ;
+                           edo:attributeNature edo:DimensionalAttribute .
 
 
 ###  https://w3id.org/energy-domain/edo#Commissioning
@@ -3066,7 +3081,8 @@ edo:ExecutiveProjectRevision rdf:type owl:Class ;
 
 ###  https://w3id.org/energy-domain/edo#ExternalDiameter
 edo:ExternalDiameter rdf:type owl:Class ;
-                     rdfs:subClassOf edo:DomainAttribute ;
+                     rdfs:subClassOf edo:DomainAttribute ,
+                     edo:Diameter ;
                      dcterms:accessRights "PUBLIC" ;
                      dcterms:identifier "ExternalDiameter" ;
                      skos:definition "Distance measured across the outermost points, passing through the center."@en ;
@@ -3075,7 +3091,11 @@ edo:ExternalDiameter rdf:type owl:Class ;
                      edo:hasAttributeScope edo:TypeLevelAttribute ;
                      edo:hasLifecycleCreationPhase edo:DetailedDesign ;
                      edo:hasTypedValue edo:FloatValue ;
-                     edo:hasValueCardinality edo:SingleValue .
+                     edo:hasValueCardinality edo:SingleValue ;
+                     edo:specializesAttribute edo:Diameter ;
+                     edo:hasUnit unit:M ;
+                     edo:lifecycleRole edo:DesignRole ;
+                     edo:attributeNature edo:DimensionalAttribute .
 
 
 ###  https://w3id.org/energy-domain/edo#ExternalReference
@@ -3775,7 +3795,8 @@ edo:GeothermalElectricity rdf:type owl:Class ;
 
 ###  https://w3id.org/energy-domain/edo#GrooveDiameter
 edo:GrooveDiameter rdf:type owl:Class ;
-                   rdfs:subClassOf edo:DomainAttribute ;
+                   rdfs:subClassOf edo:DomainAttribute ,
+                   edo:Diameter ;
                    dcterms:accessRights "PUBLIC" ;
                    dcterms:identifier "GrooveDiameter" ;
                    skos:definition "Specifies the diameter of the end fitting at the location where the groove is machined. This measurement is crucial for ensuring that the groove correctly accommodates the end fitting during installation on a vessel. The part diameter at the groove location ensures proper alignment and secure attachment of the end fitting to the installation equipment or support system."@en ;
@@ -3784,7 +3805,11 @@ edo:GrooveDiameter rdf:type owl:Class ;
                    edo:hasAttributeScope edo:InstanceLevelAttribute ;
                    edo:hasLifecycleCreationPhase edo:DetailedDesign ;
                    edo:hasTypedValue edo:FloatValue ;
-                   edo:hasValueCardinality edo:SingleValue .
+                   edo:hasValueCardinality edo:SingleValue ;
+                   edo:specializesAttribute edo:Diameter ;
+                   edo:hasUnit unit:M ;
+                   edo:lifecycleRole edo:DesignRole ;
+                   edo:attributeNature edo:DimensionalAttribute .
 
 
 ###  https://w3id.org/energy-domain/edo#GrooveHeight
@@ -3838,7 +3863,8 @@ edo:GroovePoint rdf:type owl:Class ;
 
 ###  https://w3id.org/energy-domain/edo#GrooveSupportSurfaceDiameter
 edo:GrooveSupportSurfaceDiameter rdf:type owl:Class ;
-                                 rdfs:subClassOf edo:DomainAttribute ;
+                                 rdfs:subClassOf edo:DomainAttribute ,
+                                 edo:Diameter ;
                                  dcterms:accessRights "PUBLIC" ;
                                  dcterms:identifier "GrooveSupportSurfaceDiameter" ;
                                  skos:definition "Indicates the diameter of the support surface within the groove. This measurement is critical for ensuring that the groove provides sufficient contact area for the components it interfaces with, contributing to a stable and secure fit."@en ;
@@ -3847,7 +3873,11 @@ edo:GrooveSupportSurfaceDiameter rdf:type owl:Class ;
                                  edo:hasAttributeScope edo:InstanceLevelAttribute ;
                                  edo:hasLifecycleCreationPhase edo:DetailedDesign ;
                                  edo:hasTypedValue edo:FloatValue ;
-                                 edo:hasValueCardinality edo:SingleValue .
+                                 edo:hasValueCardinality edo:SingleValue ;
+                                 edo:specializesAttribute edo:Diameter ;
+                                 edo:hasUnit unit:M ;
+                                 edo:lifecycleRole edo:DesignRole ;
+                                 edo:attributeNature edo:DimensionalAttribute .
 
 
 ###  https://w3id.org/energy-domain/edo#GrooveSupportSurfaceFilletRadius
@@ -4534,7 +4564,8 @@ edo:IntermediateValuesStrategy rdf:type owl:Class ;
 
 ###  https://w3id.org/energy-domain/edo#InternalDiameter
 edo:InternalDiameter rdf:type owl:Class ;
-                     rdfs:subClassOf edo:DomainAttribute ;
+                     rdfs:subClassOf edo:DomainAttribute ,
+                     edo:Diameter ;
                      dcterms:accessRights "PUBLIC" ;
                      dcterms:identifier "InternalDiameter" ;
                      skos:definition "Distance measured across the innermost points, passing through the center."@en ;
@@ -4545,7 +4576,11 @@ edo:InternalDiameter rdf:type owl:Class ;
                      edo:hasLifecycleCreationPhase edo:BasicDesign ,
                                                    edo:DetailedDesign ;
                      edo:hasTypedValue edo:FloatValue ;
-                     edo:hasValueCardinality edo:SingleValue .
+                     edo:hasValueCardinality edo:SingleValue ;
+                     edo:specializesAttribute edo:Diameter ;
+                     edo:hasUnit unit:M ;
+                     edo:lifecycleRole edo:DesignRole ;
+                     edo:attributeNature edo:DimensionalAttribute .
 
 
 ###  https://w3id.org/energy-domain/edo#InternalIncidentalPressureCurveAttribute
@@ -4812,7 +4847,8 @@ edo:LayerGeometryName rdf:type owl:Class ;
 
 ###  https://w3id.org/energy-domain/edo#LayerGeometryProfileThickness
 edo:LayerGeometryProfileThickness rdf:type owl:Class ;
-                                  rdfs:subClassOf edo:LayerGeometryAttribute ;
+                                  rdfs:subClassOf edo:LayerGeometryAttribute ,
+                                  edo:Thickness ;
                                   dcterms:accessRights "PUBLIC" ;
                                   dcterms:identifier "LayerGeometryProfileThickness" ;
                                   skos:definition "Indicates the thickness of the layer's geometric profile. This property is important for understanding the contribution of the layer to the overall structure, particularly in terms of strength, flexibility, and resistance to external forces."@en ;
@@ -4821,7 +4857,11 @@ edo:LayerGeometryProfileThickness rdf:type owl:Class ;
                                   edo:hasAttributeScope edo:InstanceLevelAttribute ;
                                   edo:hasLifecycleCreationPhase edo:DetailedDesign ;
                                   edo:hasTypedValue edo:FloatValue ;
-                                  edo:hasValueCardinality edo:SingleValue .
+                                  edo:hasValueCardinality edo:SingleValue ;
+                                  edo:specializesAttribute edo:Thickness ;
+                                  edo:hasUnit unit:M ;
+                                  edo:lifecycleRole edo:DesignRole ;
+                                  edo:attributeNature edo:DimensionalAttribute .
 
 
 ###  https://w3id.org/energy-domain/edo#LayerGeometryProfileWidth
@@ -4840,7 +4880,8 @@ edo:LayerGeometryProfileWidth rdf:type owl:Class ;
 
 ###  https://w3id.org/energy-domain/edo#LayerGeometryThickness
 edo:LayerGeometryThickness rdf:type owl:Class ;
-                           rdfs:subClassOf edo:LayerGeometryAttribute ;
+                           rdfs:subClassOf edo:LayerGeometryAttribute ,
+                           edo:Thickness ;
                            dcterms:accessRights "PUBLIC" ;
                            dcterms:identifier "LayerGeometryThickness" ;
                            skos:definition "Indicates the overall thickness of the layer's geometry. This property is critical for evaluating the structural contribution of the layer, including its strength, flexibility, and resistance to external forces."@en ;
@@ -4849,12 +4890,17 @@ edo:LayerGeometryThickness rdf:type owl:Class ;
                            edo:hasAttributeScope edo:InstanceLevelAttribute ;
                            edo:hasLifecycleCreationPhase edo:DetailedDesign ;
                            edo:hasTypedValue edo:FloatValue ;
-                           edo:hasValueCardinality edo:SingleValue .
+                           edo:hasValueCardinality edo:SingleValue ;
+                           edo:specializesAttribute edo:Thickness ;
+                           edo:hasUnit unit:M ;
+                           edo:lifecycleRole edo:DesignRole ;
+                           edo:attributeNature edo:DimensionalAttribute .
 
 
 ###  https://w3id.org/energy-domain/edo#LayerThickness
 edo:LayerThickness rdf:type owl:Class ;
-                   rdfs:subClassOf edo:DomainAttribute ;
+                   rdfs:subClassOf edo:DomainAttribute ,
+                   edo:Thickness ;
                    dcterms:accessRights "PUBLIC" ;
                    dcterms:identifier "LayerThickness" ;
                    skos:definition "Indicates the thickness of the layer, providing a key dimension that affects the overall strength, flexibility, and performance of the structure. This value is critical for ensuring that the layer meets design requirements and operational conditions."@en ;
@@ -4863,7 +4909,11 @@ edo:LayerThickness rdf:type owl:Class ;
                    edo:hasAttributeScope edo:InstanceLevelAttribute ;
                    edo:hasLifecycleCreationPhase edo:DetailedDesign ;
                    edo:hasTypedValue edo:FloatValue ;
-                   edo:hasValueCardinality edo:SingleValue .
+                   edo:hasValueCardinality edo:SingleValue ;
+                   edo:specializesAttribute edo:Thickness ;
+                   edo:hasUnit unit:M ;
+                   edo:lifecycleRole edo:DesignRole ;
+                   edo:attributeNature edo:DimensionalAttribute .
 
 
 ###  https://w3id.org/energy-domain/edo#LayerWatertight
@@ -6278,7 +6328,8 @@ edo:MultiValue rdf:type owl:Class ;
 
 ###  https://w3id.org/energy-domain/edo#NominalDiameter
 edo:NominalDiameter rdf:type owl:Class ;
-                    rdfs:subClassOf edo:DomainAttribute ;
+                    rdfs:subClassOf edo:DomainAttribute ,
+                    edo:Diameter ;
                     dcterms:accessRights "PUBLIC" ;
                     dcterms:identifier "NominalDiameter" ;
                     skos:definition "The standard diameter of the component or pipe, typically used for classification and specification purposes."@en ;
@@ -6287,7 +6338,10 @@ edo:NominalDiameter rdf:type owl:Class ;
                     edo:hasAttributeScope edo:InstanceLevelAttribute ;
                     edo:hasLifecycleCreationPhase edo:DetailedDesign ;
                     edo:hasTypedValue edo:StringValue ;
-                    edo:hasValueCardinality edo:SingleValue .
+                    edo:hasValueCardinality edo:SingleValue ;
+                    edo:specializesAttribute edo:Diameter ;
+                    edo:lifecycleRole edo:DesignRole ;
+                    edo:attributeNature edo:DimensionalAttribute .
 
 
 ###  https://w3id.org/energy-domain/edo#NominalFloatDensity
@@ -6749,7 +6803,8 @@ edo:OpticalFiberCable rdf:type owl:Class ;
 
 ###  https://w3id.org/energy-domain/edo#OuterDiameter
 edo:OuterDiameter rdf:type owl:Class ;
-                  rdfs:subClassOf edo:DomainAttribute ;
+                  rdfs:subClassOf edo:DomainAttribute ,
+                  edo:Diameter ;
                   dcterms:accessRights "PUBLIC" ;
                   dcterms:identifier "OuterDiameter" ;
                   skos:definition "Represents the external diameter of a component, typically measured from one outer edge to the opposite outer edge. This property is important for ensuring proper fit, compatibility, and performance within a system or assembly."@en ;
@@ -6758,7 +6813,11 @@ edo:OuterDiameter rdf:type owl:Class ;
                   edo:hasAttributeScope edo:TypeLevelAttribute ;
                   edo:hasLifecycleCreationPhase edo:DetailedDesign ;
                   edo:hasTypedValue edo:FloatValue ;
-                  edo:hasValueCardinality edo:SingleValue .
+                  edo:hasValueCardinality edo:SingleValue ;
+                  edo:specializesAttribute edo:Diameter ;
+                  edo:hasUnit unit:M ;
+                  edo:lifecycleRole edo:DesignRole ;
+                  edo:attributeNature edo:DimensionalAttribute .
 
 
 ###  https://w3id.org/energy-domain/edo#OutsideTemperature
@@ -7695,7 +7754,8 @@ edo:RingGasketMaterial rdf:type owl:Class ;
 
 ###  https://w3id.org/energy-domain/edo#RingGasketOuterDiameter
 edo:RingGasketOuterDiameter rdf:type owl:Class ;
-                            rdfs:subClassOf edo:RingGasketAttribute ;
+                            rdfs:subClassOf edo:RingGasketAttribute ,
+                            edo:Diameter ;
                             dcterms:accessRights "PUBLIC" ;
                             dcterms:identifier "RingGasketOuterDiameter" ;
                             skos:definition "The external diameter, which determines the size of the surface it seals against. This dimension is essential for providing a secure seal and preventing leakage."@en ;
@@ -7705,7 +7765,11 @@ edo:RingGasketOuterDiameter rdf:type owl:Class ;
                                                   edo:TypeLevelAttribute ;
                             edo:hasLifecycleCreationPhase edo:DetailedDesign ;
                             edo:hasTypedValue edo:FloatValue ;
-                            edo:hasValueCardinality edo:SingleValue .
+                            edo:hasValueCardinality edo:SingleValue ;
+                            edo:specializesAttribute edo:Diameter ;
+                            edo:hasUnit unit:M ;
+                            edo:lifecycleRole edo:DesignRole ;
+                            edo:attributeNature edo:DimensionalAttribute .
 
 
 ###  https://w3id.org/energy-domain/edo#RingGasketPressureRating
@@ -8073,7 +8137,8 @@ edo:Shackle rdf:type owl:Class ;
 
 ###  https://w3id.org/energy-domain/edo#ShackleInnerDiameter
 edo:ShackleInnerDiameter rdf:type owl:Class ;
-                         rdfs:subClassOf edo:DomainAttribute ;
+                         rdfs:subClassOf edo:DomainAttribute ,
+                         edo:Diameter ;
                          dcterms:accessRights "PUBLIC" ;
                          dcterms:identifier "ShackleInnerDiameter" ;
                          skos:definition "Indicates the inner diameter of the shackle. This property measures the width of the space inside the shackle, which is important for ensuring compatibility with other components and for determining the appropriate fit and load distribution."@en ;
@@ -8082,7 +8147,11 @@ edo:ShackleInnerDiameter rdf:type owl:Class ;
                          edo:hasAttributeScope edo:InstanceLevelAttribute ;
                          edo:hasLifecycleCreationPhase edo:DetailedDesign ;
                          edo:hasTypedValue edo:FloatValue ;
-                         edo:hasValueCardinality edo:SingleValue .
+                         edo:hasValueCardinality edo:SingleValue ;
+                         edo:specializesAttribute edo:Diameter ;
+                         edo:hasUnit unit:M ;
+                         edo:lifecycleRole edo:DesignRole ;
+                         edo:attributeNature edo:DimensionalAttribute .
 
 
 ###  https://w3id.org/energy-domain/edo#ShackleOpeningWidth
@@ -8101,7 +8170,8 @@ edo:ShackleOpeningWidth rdf:type owl:Class ;
 
 ###  https://w3id.org/energy-domain/edo#ShackleOuterDiameter
 edo:ShackleOuterDiameter rdf:type owl:Class ;
-                         rdfs:subClassOf edo:DomainAttribute ;
+                         rdfs:subClassOf edo:DomainAttribute ,
+                         edo:Diameter ;
                          dcterms:accessRights "PUBLIC" ;
                          dcterms:identifier "ShackleOuterDiameter" ;
                          skos:definition "Indicates the outer diameter of the shackle. This property provides the measurement of the overall width of the shackle, which is crucial for ensuring it fits properly with other components and meets the design specifications for load-bearing and connection requirements."@en ;
@@ -8110,12 +8180,17 @@ edo:ShackleOuterDiameter rdf:type owl:Class ;
                          edo:hasAttributeScope edo:InstanceLevelAttribute ;
                          edo:hasLifecycleCreationPhase edo:DetailedDesign ;
                          edo:hasTypedValue edo:FloatValue ;
-                         edo:hasValueCardinality edo:SingleValue .
+                         edo:hasValueCardinality edo:SingleValue ;
+                         edo:specializesAttribute edo:Diameter ;
+                         edo:hasUnit unit:M ;
+                         edo:lifecycleRole edo:DesignRole ;
+                         edo:attributeNature edo:DimensionalAttribute .
 
 
 ###  https://w3id.org/energy-domain/edo#ShackleThickness
 edo:ShackleThickness rdf:type owl:Class ;
-                     rdfs:subClassOf edo:DomainAttribute ;
+                     rdfs:subClassOf edo:DomainAttribute ,
+                     edo:Thickness ;
                      dcterms:accessRights "PUBLIC" ;
                      dcterms:identifier "ShackleThickness" ;
                      skos:definition "Specifies the thickness of the shackle material. This property measures the distance between the inner and outer surfaces of the shackle, which is essential for understanding its strength, durability, and ability to handle load stresses."@en ;
@@ -8124,7 +8199,11 @@ edo:ShackleThickness rdf:type owl:Class ;
                      edo:hasAttributeScope edo:InstanceLevelAttribute ;
                      edo:hasLifecycleCreationPhase edo:DetailedDesign ;
                      edo:hasTypedValue edo:FloatValue ;
-                     edo:hasValueCardinality edo:SingleValue .
+                     edo:hasValueCardinality edo:SingleValue ;
+                     edo:specializesAttribute edo:Thickness ;
+                     edo:hasUnit unit:M ;
+                     edo:lifecycleRole edo:DesignRole ;
+                     edo:attributeNature edo:DimensionalAttribute .
 
 
 ###  https://w3id.org/energy-domain/edo#ShearableRiserJoint
@@ -10070,7 +10149,8 @@ edo:UnifilarDiagramRev rdf:type owl:Class ;
 
 ###  https://w3id.org/energy-domain/edo#UpperItubeDiameter
 edo:UpperItubeDiameter rdf:type owl:Class ;
-                       rdfs:subClassOf edo:DomainAttribute ;
+                       rdfs:subClassOf edo:DomainAttribute ,
+                       edo:Diameter ;
                        dcterms:accessRights "PUBLIC" ;
                        dcterms:identifier "UpperItubeDiameter" ;
                        skos:definition "Diameter of the upper section of the tube or pipe. This measurement is important for ensuring proper fit and compatibility with other components or connections. The diameter of the upper section affects the alignment and sealing capabilities of the assembly, and ensures that the tube or pipe meets the design and operational requirements for the installation or application."@en ;
@@ -10079,7 +10159,11 @@ edo:UpperItubeDiameter rdf:type owl:Class ;
                        edo:hasAttributeScope edo:TypeLevelAttribute ;
                        edo:hasLifecycleCreationPhase edo:DetailedDesign ;
                        edo:hasTypedValue edo:FloatValue ;
-                       edo:hasValueCardinality edo:SingleValue .
+                       edo:hasValueCardinality edo:SingleValue ;
+                       edo:specializesAttribute edo:Diameter ;
+                       edo:hasUnit unit:M ;
+                       edo:lifecycleRole edo:DesignRole ;
+                       edo:attributeNature edo:DimensionalAttribute .
 
 
 ###  https://w3id.org/energy-domain/edo#Upstream
@@ -10397,7 +10481,8 @@ edo:WireRopeSegment rdf:type owl:Class ;
 
 ###  https://w3id.org/energy-domain/edo#WireRopeSlingDiameter
 edo:WireRopeSlingDiameter rdf:type owl:Class ;
-                          rdfs:subClassOf edo:DomainAttribute ;
+                          rdfs:subClassOf edo:DomainAttribute ,
+                          edo:Diameter ;
                           dcterms:accessRights "PUBLIC" ;
                           dcterms:identifier "WireRopeSlingDiameter" ;
                           skos:definition "Diâmetro do Estrobo de Corda de Aço"@pt-br ,
@@ -10407,7 +10492,11 @@ edo:WireRopeSlingDiameter rdf:type owl:Class ;
                           edo:hasAttributeScope edo:TypeLevelAttribute ;
                           edo:hasLifecycleCreationPhase edo:DetailedDesign ;
                           edo:hasTypedValue edo:FloatValue ;
-                          edo:hasValueCardinality edo:SingleValue .
+                          edo:hasValueCardinality edo:SingleValue ;
+                          edo:specializesAttribute edo:Diameter ;
+                          edo:hasUnit unit:M ;
+                          edo:lifecycleRole edo:DesignRole ;
+                          edo:attributeNature edo:DimensionalAttribute .
 
 
 ###  https://w3id.org/energy-domain/edo#WireRopeSlingLength
@@ -10729,3 +10818,186 @@ edo:RemainingFatigueLife rdf:type owl:Class ;
                          edo:attributeNature edo:MechanicalAttribute ,
                                              edo:PerformanceAttribute .
 
+###  https://w3id.org/energy-domain/edo#Diameter
+edo:Diameter rdf:type owl:Class ;
+             rdfs:subClassOf edo:DomainAttribute ;
+             dcterms:accessRights "PUBLIC" ;
+             dcterms:identifier "Diameter" ;
+             skos:definition "Generic attribute for diameter values. Specialized attributes must be used when the value represents a nominal, internal, external, required, designed, certified, as-built or another lifecycle-specific meaning."@en ,
+                             "Atributo genérico para valores de diâmetro. Atributos especializados devem ser usados quando o valor representar diâmetro nominal, interno, externo, requerido, de projeto, certificado, as-built ou outro significado específico do ciclo de vida."@pt-br ;
+             skos:prefLabel "Diameter"@en ,
+                            "Diâmetro"@pt-br ;
+             edo:hasAttributeScope edo:TypeLevelAttribute ;
+             edo:hasLifecycleCreationPhase edo:DetailedDesign ;
+             edo:hasTypedValue edo:FloatValue ;
+             edo:hasUnit unit:M ;
+             edo:hasValueCardinality edo:SingleValue ;
+             edo:lifecycleRole edo:DesignRole ;
+             edo:attributeNature edo:DimensionalAttribute .
+
+
+###  https://w3id.org/energy-domain/edo#RequiredDiameter
+edo:RequiredDiameter rdf:type owl:Class ;
+                     rdfs:subClassOf edo:Diameter ;
+                     dcterms:accessRights "PUBLIC" ;
+                     dcterms:identifier "RequiredDiameter" ;
+                     skos:definition "Diameter required by the owner, project, contract, standard or data contract."@en ,
+                                     "Diâmetro requerido pelo proprietário, projeto, contrato, norma ou contrato de dados."@pt-br ;
+                     skos:prefLabel "Diâmetro requerido"@pt-br ,
+                                    "Required Diameter"@en ;
+                     edo:specializesAttribute edo:Diameter ;
+                     edo:hasAttributeScope edo:TypeLevelAttribute ;
+                     edo:hasLifecycleCreationPhase edo:BasicDesign ;
+                     edo:hasTypedValue edo:FloatValue ;
+                     edo:hasUnit unit:M ;
+                     edo:hasValueCardinality edo:SingleValue ;
+                     edo:lifecycleRole edo:RequirementRole ;
+                     edo:attributeNature edo:DimensionalAttribute .
+
+
+###  https://w3id.org/energy-domain/edo#DesignedDiameter
+edo:DesignedDiameter rdf:type owl:Class ;
+                     rdfs:subClassOf edo:Diameter ;
+                     dcterms:accessRights "PUBLIC" ;
+                     dcterms:identifier "DesignedDiameter" ;
+                     skos:definition "Diameter established by design for the component, layer or system."@en ,
+                                     "Diâmetro estabelecido em projeto para o componente, camada ou sistema."@pt-br ;
+                     skos:prefLabel "Designed Diameter"@en ,
+                                    "Diâmetro de projeto"@pt-br ;
+                     edo:specializesAttribute edo:Diameter ;
+                     edo:hasAttributeScope edo:TypeLevelAttribute ;
+                     edo:hasLifecycleCreationPhase edo:DetailedDesign ;
+                     edo:hasTypedValue edo:FloatValue ;
+                     edo:hasUnit unit:M ;
+                     edo:hasValueCardinality edo:SingleValue ;
+                     edo:lifecycleRole edo:DesignRole ;
+                     edo:attributeNature edo:DimensionalAttribute .
+
+
+###  https://w3id.org/energy-domain/edo#CertifiedDiameter
+edo:CertifiedDiameter rdf:type owl:Class ;
+                      rdfs:subClassOf edo:Diameter ;
+                      dcterms:accessRights "PUBLIC" ;
+                      dcterms:identifier "CertifiedDiameter" ;
+                      skos:definition "Diameter certified for the component, layer or system after verification, inspection, testing, certification or supplier declaration."@en ,
+                                      "Diâmetro certificado para o componente, camada ou sistema após verificação, inspeção, ensaio, certificação ou declaração do fornecedor."@pt-br ;
+                      skos:prefLabel "Certified Diameter"@en ,
+                                     "Diâmetro certificado"@pt-br ;
+                      edo:specializesAttribute edo:Diameter ;
+                      edo:hasAttributeScope edo:TypeLevelAttribute ;
+                      edo:hasTypedValue edo:FloatValue ;
+                      edo:hasUnit unit:M ;
+                      edo:hasValueCardinality edo:SingleValue ;
+                      edo:lifecycleRole edo:CertifiedRole ;
+                      edo:attributeNature edo:DimensionalAttribute .
+
+
+###  https://w3id.org/energy-domain/edo#AsBuiltDiameter
+edo:AsBuiltDiameter rdf:type owl:Class ;
+                    rdfs:subClassOf edo:Diameter ;
+                    dcterms:accessRights "PUBLIC" ;
+                    dcterms:identifier "AsBuiltDiameter" ;
+                    skos:definition "Diameter recorded for the component, layer or system as built or manufactured."@en ,
+                                    "Diâmetro registrado para o componente, camada ou sistema como construído ou fabricado."@pt-br ;
+                    skos:prefLabel "As-Built Diameter"@en ,
+                                   "Diâmetro as-built"@pt-br ;
+                    edo:specializesAttribute edo:Diameter ;
+                    edo:hasAttributeScope edo:InstanceLevelAttribute ;
+                    edo:hasTypedValue edo:FloatValue ;
+                    edo:hasUnit unit:M ;
+                    edo:hasValueCardinality edo:SingleValue ;
+                    edo:lifecycleRole edo:AsBuiltRole ;
+                    edo:attributeNature edo:DimensionalAttribute .
+
+
+###  https://w3id.org/energy-domain/edo#Thickness
+edo:Thickness rdf:type owl:Class ;
+              rdfs:subClassOf edo:DomainAttribute ;
+              dcterms:accessRights "PUBLIC" ;
+              dcterms:identifier "Thickness" ;
+              skos:definition "Generic attribute for thickness values. Specialized attributes must be used when the value represents a layer, geometry, required, designed, certified, as-built or another lifecycle-specific meaning."@en ,
+                              "Atributo genérico para valores de espessura. Atributos especializados devem ser usados quando o valor representar espessura de camada, geometria, requerida, de projeto, certificada, as-built ou outro significado específico do ciclo de vida."@pt-br ;
+              skos:prefLabel "Espessura"@pt-br ,
+                             "Thickness"@en ;
+              edo:hasAttributeScope edo:TypeLevelAttribute ;
+              edo:hasLifecycleCreationPhase edo:DetailedDesign ;
+              edo:hasTypedValue edo:FloatValue ;
+              edo:hasUnit unit:M ;
+              edo:hasValueCardinality edo:SingleValue ;
+              edo:lifecycleRole edo:DesignRole ;
+              edo:attributeNature edo:DimensionalAttribute .
+
+
+###  https://w3id.org/energy-domain/edo#RequiredThickness
+edo:RequiredThickness rdf:type owl:Class ;
+                      rdfs:subClassOf edo:Thickness ;
+                      dcterms:accessRights "PUBLIC" ;
+                      dcterms:identifier "RequiredThickness" ;
+                      skos:definition "Thickness required by the owner, project, contract, standard or data contract."@en ,
+                                      "Espessura requerida pelo proprietário, projeto, contrato, norma ou contrato de dados."@pt-br ;
+                      skos:prefLabel "Espessura requerida"@pt-br ,
+                                     "Required Thickness"@en ;
+                      edo:specializesAttribute edo:Thickness ;
+                      edo:hasAttributeScope edo:TypeLevelAttribute ;
+                      edo:hasLifecycleCreationPhase edo:BasicDesign ;
+                      edo:hasTypedValue edo:FloatValue ;
+                      edo:hasUnit unit:M ;
+                      edo:hasValueCardinality edo:SingleValue ;
+                      edo:lifecycleRole edo:RequirementRole ;
+                      edo:attributeNature edo:DimensionalAttribute .
+
+
+###  https://w3id.org/energy-domain/edo#DesignedThickness
+edo:DesignedThickness rdf:type owl:Class ;
+                      rdfs:subClassOf edo:Thickness ;
+                      dcterms:accessRights "PUBLIC" ;
+                      dcterms:identifier "DesignedThickness" ;
+                      skos:definition "Thickness established by design for the component, layer or system."@en ,
+                                      "Espessura estabelecida em projeto para o componente, camada ou sistema."@pt-br ;
+                      skos:prefLabel "Designed Thickness"@en ,
+                                     "Espessura de projeto"@pt-br ;
+                      edo:specializesAttribute edo:Thickness ;
+                      edo:hasAttributeScope edo:TypeLevelAttribute ;
+                      edo:hasLifecycleCreationPhase edo:DetailedDesign ;
+                      edo:hasTypedValue edo:FloatValue ;
+                      edo:hasUnit unit:M ;
+                      edo:hasValueCardinality edo:SingleValue ;
+                      edo:lifecycleRole edo:DesignRole ;
+                      edo:attributeNature edo:DimensionalAttribute .
+
+
+###  https://w3id.org/energy-domain/edo#CertifiedThickness
+edo:CertifiedThickness rdf:type owl:Class ;
+                       rdfs:subClassOf edo:Thickness ;
+                       dcterms:accessRights "PUBLIC" ;
+                       dcterms:identifier "CertifiedThickness" ;
+                       skos:definition "Thickness certified for the component, layer or system after verification, inspection, testing, certification or supplier declaration."@en ,
+                                       "Espessura certificada para o componente, camada ou sistema após verificação, inspeção, ensaio, certificação ou declaração do fornecedor."@pt-br ;
+                       skos:prefLabel "Certified Thickness"@en ,
+                                      "Espessura certificada"@pt-br ;
+                       edo:specializesAttribute edo:Thickness ;
+                       edo:hasAttributeScope edo:TypeLevelAttribute ;
+                       edo:hasTypedValue edo:FloatValue ;
+                       edo:hasUnit unit:M ;
+                       edo:hasValueCardinality edo:SingleValue ;
+                       edo:lifecycleRole edo:CertifiedRole ;
+                       edo:attributeNature edo:DimensionalAttribute .
+
+
+###  https://w3id.org/energy-domain/edo#AsBuiltThickness
+edo:AsBuiltThickness rdf:type owl:Class ;
+                     rdfs:subClassOf edo:Thickness ;
+                     dcterms:accessRights "PUBLIC" ;
+                     dcterms:identifier "AsBuiltThickness" ;
+                     skos:definition "Thickness recorded for the component, layer or system as built or manufactured."@en ,
+                                     "Espessura registrada para o componente, camada ou sistema como construído ou fabricado."@pt-br ;
+                     skos:prefLabel "As-Built Thickness"@en ,
+                                    "Espessura as-built"@pt-br ;
+                     edo:specializesAttribute edo:Thickness ;
+                     edo:hasAttributeScope edo:InstanceLevelAttribute ;
+                     edo:hasTypedValue edo:FloatValue ;
+                     edo:hasUnit unit:M ;
+                     edo:hasValueCardinality edo:SingleValue ;
+                     edo:lifecycleRole edo:AsBuiltRole ;
+                     edo:attributeNature edo:DimensionalAttribute .
+
