diff --git a/modules/alignments/ido/edo-ido-alignment.ttl b/modules/alignments/ido/edo-ido-alignment.ttl
index 140497a..3bccca3 100644
--- a/modules/alignments/ido/edo-ido-alignment.ttl
+++ b/modules/alignments/ido/edo-ido-alignment.ttl
@@ -7,6 +7,9 @@
 
 <https://w3id.org/energy-domain/edo/alignments/ido>
     a owl:Ontology ;
+    owl:imports
+        <https://w3id.org/energy-domain/edo> ,
+        <http://rds.posccaesar.org/ontology/lis14/ont/core> ;
     rdfs:label "EDO to IDO Alignment" ;
     dcterms:description """
     Alignment module relating the Energy Domain Ontology (EDO)
@@ -80,9 +83,6 @@ edo:Port
 edo:DomainAttribute
     skos:closeMatch lis:Quality .
 
-edo:Pressure
-    skos:closeMatch lis:PhysicalQuantity .
-
 edo:WaterDepth
     skos:closeMatch lis:PhysicalQuantity .
 
@@ -101,6 +101,315 @@ edo:CertifiedMaxWaterDepth
 edo:InstalledWaterDepth
     skos:closeMatch lis:PhysicalQuantity .
 
+edo:Pressure
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:MaxPressure
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:RequiredMaxPressure
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:MaxDesignPressure
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:CertifiedMaxPressure
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:OperatingPressure
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:Temperature
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:MaximumTemperature
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:MinimumTemperature
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:RequiredMaximumTemperature
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:DesignedMaximumTemperature
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:CertifiedMaximumTemperature
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:OperatingTemperature
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:RequiredMinimumTemperature
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:DesignedMinimumTemperature
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:CertifiedMinimumTemperature
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:InsideTemperature
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:OutsideTemperature
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:Length
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:NominalLength
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:RequiredLength
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:DesignedLength
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:CertifiedLength
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:AsBuiltLength
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:InstalledLength
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:Tension
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:MaxTension
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:RequiredMaxTension
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:DesignedMaxTension
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:CertifiedMaxTension
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:OperatingTension
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:SpoolingTension
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:BendingRadius
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:MinimumBendingRadius
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:RequiredMinimumBendingRadius
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:DesignedMinimumBendingRadius
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:CertifiedMinimumBendingRadius
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:OperatingBendingRadius
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:MinimumBendingRadiusForStorage
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:DesignLife
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:RequiredDesignLife
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:CertifiedDesignLife
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:RemainingDesignLife
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:FatigueLife
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:RequiredFatigueLife
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:DesignedFatigueLife
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:CertifiedFatigueLife
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:RemainingFatigueLife
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:Mass
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:RequiredMass
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:DesignedMass
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:CertifiedMass
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:AsBuiltMass
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:LinearMass
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:IndividualAnodeMass
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:Weight
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:RequiredWeight
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:DesignedWeight
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:CertifiedWeight
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:AsBuiltWeight
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:SpecificWeight
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:Diameter
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:RequiredDiameter
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:DesignedDiameter
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:CertifiedDiameter
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:AsBuiltDiameter
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:Thickness
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:RequiredThickness
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:DesignedThickness
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:CertifiedThickness
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:AsBuiltThickness
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:BurstPressure
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:RequiredBurstPressure
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:DesignedBurstPressure
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:CertifiedBurstPressure
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:OperatingBurstPressure
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:CalculatedAbsoluteBurstPressure
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:CollapsePressure
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:RequiredCollapsePressure
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:DesignedCollapsePressure
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:CertifiedCollapsePressure
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:OperatingCollapsePressure
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:HydrostaticCollapseAbsPressDry
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:Volume
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:RequiredVolume
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:DesignedVolume
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:CertifiedVolume
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:AsBuiltVolume
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:InternalVolume
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:RequiredInternalVolume
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:DesignedInternalVolume
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:CertifiedInternalVolume
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:AsBuiltInternalVolume
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:DisplacedVolume
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:RequiredDisplacedVolume
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:DesignedDisplacedVolume
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:CertifiedDisplacedVolume
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:AsBuiltDisplacedVolume
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:Distance
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:Offset
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:LongitudinalOffset
+    skos:closeMatch lis:PhysicalQuantity .
+
+edo:Position
+    skos:relatedMatch lis:Location .
+
+edo:PipeMountPosition
+    skos:relatedMatch lis:Location .
+
 #################################################################
 #    Attribute classification schemes
 #################################################################
